# doi_10_3390-toxics11121001

**Source PDF**: doi_10_3390-toxics11121001.pdf

---

## Page 1

Next Article in Journal
Reduced Isocyanate Release Using a Waterproof, Resin-Based Cast
Alternative Relative to Fiberglass Casts
Next Article in Special Issue
In Vitro and In Silico Analysis of the Bindings between Legacy and
Novel Per- and Polyfluoroalkyl Substances and Human Serum
Albumin
Previous Article in Journal
Invisible Hand behind Female Reproductive Disorders: Bisphenols,
Recent Evidence and Future Perspectives
Previous Article in Special Issue
The Association of Perfluoroalkyl Substance Exposure and a Serum
Liver Function Marker in Korean Adults
Journals
Active Journals Find a Journal Journal Proposal Proceedings Series
Topics
Information
For Authors For Reviewers For Editors For Librarians For Publishers
For Societies For Conference Organizers
Open Access Policy Institutional Open Access Program Special
Issues Guidelines Editorial Process Research and Publication Ethics
Article Processing Charges Awards Testimonials
Author Services
Initiatives
Sciforum MDPI Books Preprints.org Scilit SciProfiles Encyclopedia
JAMS Proceedings Series
About

## Page 2

Overview Contact Careers News Press Blog
Sign In / Sign Up
Notice
clear
Notice
You are accessing a machine-readable page. In order to be humanreadable, please install an RSS reader.
Continue Cancel
clear
All articles published by MDPI are made immediately available
worldwide under an open access license. No special permission is
required to reuse all or part of the article published by MDPI,
including figures and tables. For articles published under an open
access Creative Common CC BY license, any part of the article may
be reused without permission provided that the original article is
clearly cited. For more information, please refer to https://
www.mdpi.com/openaccess.
Feature papers represent the most advanced research with
significant potential for high impact in the field. A Feature Paper
should be a substantial original Article that involves several
techniques or approaches, provides an outlook for future research
directions and describes possible research applications.
Feature papers are submitted upon individual invitation or
recommendation by the scientific editors and must receive positive
feedback from the reviewers.
Editor’s Choice articles are based on recommendations by the
scientific editors of MDPI journals from around the world. Editors
select a small number of articles recently published in the journal
that they believe will be particularly interesting to readers, or
important in the respective research area. The aim is to provide a

## Page 3

snapshot of some of the most exciting work published in the various
research areas of the journal.
Original Submission Date Received: .
You seem to have javascript disabled. Please note that many of the
page functionalities won't work as expected without javascript
enabled.
[image]
search menu
[image]
•  Journals
○  
■  Active Journals
■  Find a Journal
■  Journal Proposal
■  Proceedings Series
•  Topics
•  Information
○  
■  For Authors
■  For Reviewers
■  For Editors
■  For Librarians
■  For Publishers
■  For Societies
■  For Conference Organizers
■  Open Access Policy
■  Institutional Open Access Program
■  Special Issues Guidelines
■  Editorial Process
■  Research and Publication Ethics
■  Article Processing Charges
■  Awards
■  Testimonials
•  Author Services

## Page 4

•  Initiatives
○  
■  Sciforum
■  MDPI Books
■  Preprints.org
■  Scilit
■  SciProfiles
■  Encyclopedia
■  JAMS
■  Proceedings Series
•  About
○  
■  Overview
■  Contact
■  Careers
■  News
■  Press
■  Blog
Sign In / Sign Up Submit

Search for Articles:
Title / Keyword
Author / Affiliation / Email
Journal
All Journals Accounting and Auditing Acoustics Acta Microbiologica
Hellenica (AMH) Actuators Adhesives Administrative Sciences
Adolescents Advances in Respiratory Medicine (ARM) Aerobiology
Aerospace Agriculture AgriEngineering Agrochemicals Agronomy AI
AI Chemistry AI in Education AI Materials AI Sensors Air
Algorithms Allergies Alloys Analytica Analytics Anatomia
Anesthesia Research Animals Antibiotics Antibodies Antioxidants
Applied Biosciences Applied Mechanics Applied Microbiology
Applied Nano Applied Sciences Applied System Innovation (ASI)
AppliedChem AppliedMath AppliedPhys Aquaculture Journal
Architecture Arthropoda Arts Astronautics Astronomy Atmosphere

## Page 5

Atoms Audiology Research Automation Axioms Bacteria Batteries
Behavioral Sciences Beverages Big Data and Cognitive Computing
(BDCC) BioChem Bioengineering Biologics Biology Biology and Life
Sciences Forum Biomass Biomechanics BioMed Biomedicines
BioMedInformatics Biomimetics Biomolecules Biophysica
Bioresources and Bioproducts Biosensors Biosphere BioTech Birds
Blockchains Brain Sciences Buildings Businesses C (Journal of
Carbon Research) Cancers Cardiogenetics Cardiovascular Medicine
Catalysts Cells Ceramics Challenges ChemEngineering Chemistry
Chemistry Proceedings Chemosensors Children Chips CivilEng Clean
Technologies (Clean Technol.) Climate Clinical and Translational
Neuroscience (CTN) Clinical Bioenergetics Clinics and Practice
Clocks & Sleep Coasts Coatings Colloids and Interfaces Colorants
Commodities Complexities Complications Compounds Computation
Computer Sciences & Mathematics Forum Computers Condensed
Matter Conservation Construction Materials Corrosion and Materials
Degradation (CMD) Cosmetics COVID Craniomaxillofacial Trauma
& Reconstruction (CMTR) Crops Cryo Cryptography Crystals Culture
Current Issues in Molecular Biology (CIMB) Current Oncology Dairy
Data Dentistry Journal Dermato Dermatopathology Designs
Diabetology Diagnostics Dietetics Digital Disabilities Diseases
Diversity DNA Drones Drugs and Drug Candidates (DDC) Dynamics
Earth Ecologies Econometrics Economies Education Sciences
Electricity Electrochem Electronic Materials Electronics Emergency
Care and Medicine Encyclopedia Endocrines Energies Energy
Storage and Applications (ESA) Eng Engineering Proceedings
Entropic and Disordered Matter (EDM) Entropy Environmental and
Earth Sciences Proceedings Environments Epidemiologia
Epigenomes European Burn Journal (EBJ) European Journal of
Investigation in Health, Psychology and Education (EJIHPE) Family
Sciences Fermentation Fibers FinTech Fire Fishes Fluids Foods
Forecasting Forensic Sciences Forests Fossil Studies Foundations
Fractal and Fractional (Fractal Fract) Fuels Future Future Internet
Future Pharmacology Future Transportation Galaxies Games Gases
Gastroenterology Insights Gastrointestinal Disorders Gastronomy
Gels Genealogy Genes Geographies GeoHazards Geomatics
Geometry Geosciences Geotechnics Geriatrics GERMS Glacies Gout,
Urate, and Crystal Deposition Disease (GUCDD) Grasses Green
Health Hardware Healthcare Hearts Hemato Hematology Reports
Heritage Histories Horticulturae Hospitals Humanities Humans

## Page 6

Hydrobiology Hydrogen Hydrology Hydropower Hygiene Immuno
Infectious Disease Reports Informatics Information Infrastructures
Inorganics Insects Instruments Intelligent Infrastructure and
Construction International Journal of Cognitive Sciences (IJCS)
International Journal of Environmental Medicine (IJEM)
International Journal of Environmental Research and Public Health
(IJERPH) International Journal of Financial Studies (IJFS)
International Journal of Molecular Sciences (IJMS) International
Journal of Neonatal Screening (IJNS) International Journal of
Orofacial Myology and Myofunctional Therapy (IJOM) International
Journal of Plant Biology (IJPB) International Journal of Topology
International Journal of Translational Medicine (IJTM)
International Journal of Turbomachinery, Propulsion and Power
(IJTPP) International Medical Education (IME) Inventions IoT ISPRS
International Journal of Geo-Information (IJGI) J Journal of
Aesthetic Medicine (J. Aesthetic Med.) Journal of Ageing and
Longevity (JAL) Journal of CardioRenal Medicine (JCRM) Journal
of Cardiovascular Development and Disease (JCDD) Journal of
Clinical & Translational Ophthalmology (JCTO) Journal of Clinical
Medicine (JCM) Journal of Composites Science (J. Compos. Sci.)
Journal of Cybersecurity and Privacy (JCP) Journal of Dementia
and Alzheimer's Disease (JDAD) Journal of Developmental Biology
(JDB) Journal of Experimental and Theoretical Analyses (JETA)
Journal of Eye Movement Research (JEMR) Journal of Functional
Biomaterials (JFB) Journal of Functional Morphology and
Kinesiology (JFMK) Journal of Fungi (JoF) Journal of Imaging (J.
Imaging) Journal of Intelligence (J. Intell.) Journal of Low Power
Electronics and Applications (JLPEA) Journal of Manufacturing and
Materials Processing (JMMP) Journal of Marine Science and
Engineering (JMSE) Journal of Market Access & Health Policy
(JMAHP) Journal of Mind and Medical Sciences (JMMS) Journal of
Molecular Pathology (JMP) Journal of Nanotheranostics (JNT)
Journal of Nuclear Engineering (JNE) Journal of
Otorhinolaryngology, Hearing and Balance Medicine (JOHBM)
Journal of Parks Journal of Personalized Medicine (JPM) Journal of
Pharmaceutical and BioTech Industry (JPBI) Journal of Respiration
(JoR) Journal of Risk and Financial Management (JRFM) Journal of
Sensor and Actuator Networks (JSAN) Journal of the Oman Medical
Association (JOMA) Journal of Theoretical and Applied Electronic
Commerce Research (JTAER) Journal of Vascular Diseases (JVD)

## Page 7

Journal of Xenobiotics (JoX) Journal of Zoological and Botanical
Gardens (JZBG) Journalism and Media Kidney and Dialysis Kinases
and Phosphatases Knowledge LabMed Laboratories Land Languages
Laws Life Lights Limnological Review Lipidology Liquids Literature
Livers Logics Logistics Lubricants Lymphatics Machine Learning and
Knowledge Extraction (MAKE) Machines Macromol Magnetism
Magnetochemistry Marine Drugs Materials Materials Proceedings
Mathematical and Computational Applications (MCA) Mathematics
Medical Sciences Medical Sciences Forum Medicina Medicines
Membranes Merits Metabolites Metals Meteorology Methane
Methods and Protocols (MPs) Metrics Metrology Micro
Microbiology Research Microelectronics Micromachines
Microorganisms Microplastics Microwave Minerals Mining
Modelling Modern Mathematical Physics Molbank Molecules
Multimedia Multimodal Technologies and Interaction (MTI) Muscles
Nanoenergy Advances Nanomanufacturing Nanomaterials NDT
Network Neuroglia Neurology International NeuroSci Nitrogen NonCoding RNA (ncRNA) Nursing Reports Nutraceuticals Nutrients
Obesities Occupational Health Oceans Onco Optics Oral Organics
Organoids Osteology Oxygen Parasitologia Particles Pathogens
Pathophysiology Peace Studies Pediatric Reports Pets
Pharmaceuticals Pharmaceutics Pharmacoepidemiology Pharmacy
Philosophies Photochem Photonics Phycology Physchem Physical
Sciences Forum Physics Physiologia Plants Plasma Platforms
Pollutants Polymers Polysaccharides Populations Poultry Powders
Precision Oncology Proceedings Processes Prosthesis Proteomes
Psychiatry International Psychoactives Psychology International
Publications Purification Quantum Beam Science (QuBS) Quantum
Reports Quaternary Radiation Reactions Real Estate Receptors
Recycling Regional Science and Environmental Economics (RSEE)
Religions Remote Sensing Reports Reproductive Medicine (Reprod.
Med.) Resources Rheumato Risks Robotics Romanian Journal of
Preventive Medicine (RJPM) Ruminants Safety Sci Scientia
Pharmaceutica (Sci. Pharm.) Sclerosis Seeds Sensors Separations
Sexes Signals Sinusitis Smart Cities Social Sciences Société
Internationale d’Urologie Journal (SIUJ) Societies Software Soil
Systems Solar Solids Spectroscopy Journal Sports Standards Stats
Stresses Surfaces Surgeries Surgical Techniques Development
Sustainability Sustainable Chemistry Symmetry SynBio Systems
Targets Taxonomy Technologies Telecom Textiles Thalassemia

## Page 8

Reports Theoretical and Applied Ergonomics Therapeutics Thermo
Time and Space Tomography Tourism and Hospitality Toxics Toxins
Transplantology Trauma Care Trends in Higher Education Tropical
Medicine and Infectious Disease (TropicalMed) Universe Urban
Science Uro Vaccines Vehicles Venereology Veterinary Sciences
Vibration Virtual Worlds Viruses Vision Waste Water Wild Wind
Women World World Electric Vehicle Journal (WEVJ) Youth
Zoonotic Diseases
Article Type
All Article Types Article Review Communication Editorial Abstract
Book Review Brief Communication Brief Report Case Report
Clinicopathological Challenge Comment Commentary Concept
Paper Conference Report Correction Creative Data Descriptor
Discussion Entry Essay Expression of Concern Extended Abstract
Field Guide Giants in Urology Guidelines Hypothesis Interesting
Images Letter New Book Received Obituary Opinion Patent
Summary Perspective Proceeding Paper Project Report Protocol
Registered Report Reply Retraction Short Note Study Protocol
Systematic Review Technical Note Tutorial Urology around the
World Viewpoint

Advanced Search
 
Section
Special Issue
Volume
Issue
Number
Page
 
Logical OperatorOperator

## Page 9

AND OR
Search Text
Search Type
All fields Title Abstract Keywords Authors Affiliations Doi Full Text
References
 
add_circle_outline
remove_circle_outline

Journals
Toxics
Volume 11
Issue 12
10.3390/toxics11121001
[image]
Submit to this Journal Review for this Journal Propose a Special
Issue
► Article Menu
Article Menu
•  Academic Editors
[image]Chang-Gui Pan
[image]Runxia Sun
•  Subscribe SciFeed
•  Related Info Links
○  PubMed/Medline
○  Google Scholar
•  More by Authors Links
○  on DOAJ
○  on Google Scholar
○  on PubMed
Article Views
Citations -

## Page 10

•  Table of Contents
share Share announcement Help format_quote Cite question_answer
Discuss in SciProfiles
Need Help?
Support
Find support for a specific problem in the support section of our
website.
Get Support
Feedback
Please let us know what you think of our products and services.
Give Feedback
Information
Visit our dedicated information section to learn more about MDPI.
Get Information
clear
JSmol Viewer
clear
first_page
settings
Order Article Reprints
Font Type:
Arial Georgia Verdana
Font Size:
Aa Aa Aa
Line Spacing:
  

## Page 11

Column Width:
  
Background:
Open AccessCommunication
Biodegradation Potential of C7C10 Perfluorocarboxylic Acids
and Data from the Genome of a
New Strain of
Pseudomonas mosselii 5(3)
by [image],
Sergey Chetverikov
[image]Sergey Chetverikov
SciProfiles Scilit Preprints.org Google Scholar
 *,
Gaisar Hkudaygulov
[image]Gaisar Hkudaygulov
SciProfiles Scilit Preprints.org Google Scholar
 ,
Danil Sharipov
[image]Danil Sharipov
SciProfiles Scilit Preprints.org Google Scholar
 and
Sergey Starikov
[image]Sergey Starikov
SciProfiles Scilit Preprints.org Google Scholar
Darya Chetverikova
[image]Darya Chetverikova
SciProfiles Scilit Preprints.org Google Scholar
Ufa Institute of Biology, Subdivision of the Ufa Federal Research

## Page 12

Centre of the Russian Academy of Sciences, 450054 Ufa, Russia
*
Author to whom correspondence should be addressed.
Toxics 2023, 11(12), 1001; https://doi.org/10.3390/
toxics11121001
Submission received: 3 November 2023
/
Revised: 4 December 2023
/
Accepted: 6 December 2023
/
Published: 8 December 2023
(This article belongs to the Special Issue Occurrence, Fate, Removal,
and Effects of Per- and Polyfluoroalkyl Substances (PFASs))
Download keyboard_arrow_down
Download PDF
Download PDF with Cover
Download XML
Download Epub
Download Supplementary Material
Browse Figures
Review Reports Versions Notes
:
Abstract
The use of bacteria of the genus Pseudomonas—destructors of
persistent pollutants for biotechnologies of environmental
purification—is an interesting area of research. The aim of this
work was to study the potential of Pseudomonas mosselii strain 5(3)
isolated from pesticide-contaminated soil as a degrader of C7-C10
perfluorocarboxylic acids (PFCAs) and analyze its complete
genome. The genome of the strain has been fully sequenced. It
consists of a chromosome with a length of 5,676,241 b.p. and

## Page 13

containing a total of 5134 genes, in particular, haloalkane
dehalogenase gene (dhaA), haloacetate dehalogenase H-1 gene
(dehH1), fluoride ion transporter gene (crcB) and alkanesulfonate
monooxygenase gene (ssuE), responsible for the degradation of
fluorinated compounds. The strain P. mosselii 5(3) for was
cultivated for 7 days in a liquid medium with various C7-C10 PFCAs
as the sole source of carbon and energy, and completely disposed of
them. The results of LC-MS analysis showed that the transformation
takes place due to perfluorohexanoic acid with the release of
various levels of stoichiometry (depending on PFCA) of fluorine ion
mineralization indicators determined by ion chromatography. Thus,
Pseudomonas mosselii strain 5(3) demonstrates a genetically
confirmed high potential for the decomposition of C7-C10 PFCA.
Keywords:
perfluorocarboxylic acids; biodegradation; transformation;
Pseudomonas; genome
1. Introduction
Perfluorocarboxylic acids (PFCAs) have recently been subjected to
close regulation by international supervisory authorities due to their
lability; bioaccumulation ability; and resistance, including
biostability that is based on the strength of the carbon–fluorine
interaction, which forms the strongest bond ever discovered [1].
The identification of these properties characterizing their toxicity to
wildlife led earlier to their inclusion in Annex B of the Stockholm
Convention on Persistent Organic Pollutants [2].
The main significant representatives of this class of compounds are
perfluorooctanoic acid (PFOA) and perfluorooctanesulfonate
(PFOS), these synthetic chemical compounds exhibit remarkable
stability and are extensively employed in the fabrication of
fluoropolymers for non-stick surfaces, pesticides, protective agents
for fabrics, food packaging, electroplating and as components of
fire-fighting foams, such as surfactants [3,4,5,6,7,8]. As a result,
PFCAs are omnipresent in the global ecosystem, migrating through
soil and water systems [9,10]. Its toxicological effects on humans
are characterized by various types of cancer, disorders of the

## Page 14

immune system and liver, and influence on growth, fetal
development and reproductive function [11,12,13,14].
Hence, the imperative lies in expeditiously devising effective
techniques for the decomposition of PFCA compounds. Various
approaches have been explored for the degradation of PFOS,
including chemical treatment, high-temperature incineration, and
ultrasound techniques. However, these methods have proven to be
both costly and inefficient in achieving satisfactory results
[15,16,17,18]. The biodegradation of PFCAs is further complicated
by their unique chemical properties, specifically their intricate
molecular structures and substantial molecular sizes which pose a
myriad of physiological challenges for microorganisms, the
absorption of which by bacteria requires specialized active
transport pathways. Previously, it was assumed that
microorganisms did not have time to evolve in order to decompose
PFCA (especially in comparison with the favorable results of the
dehalogenation of polychlorinated molecules [19]) by developing
the enzymatic systems acting on them. This was due to the lack of
analogues in nature that would stimulate the evolution of the
necessary catabolic enzymes [20]. But recently it has been shown
that the deformation of PFCA is still possible. And it occurs mainly
when microbial enzymes attack fluoride-free carbon covalent bonds
(for example, CS and CO), as shown during the biodegradation of
PFOA and PFOS, fluorotelomers in aerobic [21,22] and anaerobic
conditions [23,24,25]. Some other works show the destruction of
PFOS and/or PFOA, which is not confirmed by the release of
fluorine ions, suggesting that minor polyfluorinated compounds are
preferably formed as a result of these biotransformations
[26,27,28,29]. The successful disruption of CF bonds has been
widely acknowledged as a pivotal factor in the natural breakdown
of organofluorine substances, such as PFCA [30]. The elimination of
fluoride effectively nullifies the discernible impact of this
heteroatom on the chemical stability of the organic compound,
which is attributable to its exceptional electronegativity. This not
only enhances the probability of the facile decomposition of the
resultant metabolites but also accentuates their overall
susceptibility to breakdown [31].
In most of these studies, the destructors of organofluorine
compounds are Pseudomonads. These microorganisms have
garnered extensive research attention owing to their remarkable

## Page 15

metabolic adaptability; they are destructors of various xenobiotics.
Harnessing their unparalleled metabolic versatility, these
microorganisms thrive by exclusively utilizing them as the primary
carbon and energy source. The genomes of various Pseudomonas
strains across distinct species has been meticulously analyzed and
documented, for example, P. protegens (strain Pf-5), P. fluorescens
(strains Pf-01, SBW25), P. azelaica (strains HBP1, Aramco J), P.
putida (strain L48), P. mendocina (strain YMP), etc. The
approximate size of Pseudomonas genomes is around 6 Mb [32]. A
multitude of genes participate in the breakdown of diverse carbon
sources and their adaptation to thrive within a specific ecological
habitat.
But from the point of view of genomic data, studies on PFCA
degradation genes are an open book that requires filling. Currently,
a scarcity of knowledge regarding bacterial strains harboring
genetic sequences responsible for encoding a diverse array of
enzymes that break down PFCAs can be detected and described.
The exceptions are P. parafulva YAB-1 and Delftia acidovorans D4B
strains (the genus Delftia is close to the genus Pseudomonas)—
exploring the potential of bacterial strains housing recognized
putative halogen acid dehalogenases and fluoroacetate
dehalogenase as catalysts in the breakdown of fluorinated
compounds [33,34]. And the list is not so extensive; only a few
strains of bacteria are known to be capable of transforming
perfluorocarboxylic acids. And they are only associated with the
biodegradation of PFOA and PFOS, with no mentions found in the
literature regarding other PFCA compounds. The aforementioned
strain P. parafulva YAB-1 can degrade 32.2% of perfluorooctanoic
acid at an initial concentration of 500 mg/L, while P. aeruginosa
HJ4 can break down PFOS by 67% at a concentration not exceeding
2 mg/L [26].
PFCA biodegradation researchers focus mainly on C8 compounds,
but their bioaccumulation potential increases with increasing
carbon chain length [35]. Recent studies have highlighted that
short-chain PFCAs exhibit enhanced mobility, increased water
solubility, and heightened resistance to degradation when compared
to their long-chain counterparts [36,37].
Therefore, when assessing biodegradation, it is necessary not to
focus only on PFOA and PFOS—the Stockholm Convention’s
inventory of persistent fluorinated organic pollutants will likely

## Page 16

expand this in the future—as the recently uncovered toxic attributes
of short-chain PFCA have emerged. Additionally, if the adaptation
of microorganisms to PFOS and PFOA has already been presented in
publications, then the resistance of microorganisms to intermediate
products of their defluorination and decomposition and released
fluorine ions, on the one hand, and their high-molecular precursors,
on the other hand, have yet to be evaluated.
In this study, a new strain called Pseudomonas mosselii 5(3) was
tested and isolated from soil contaminated with herbicides,
including fluorinated ones. This strain was identified using methods
of molecular biology, as well as the analysis of morphological,
physiological and biochemical characteristics. The genome
sequencing, the study of genetic features, and biodegradation
potential of the strain in relation to C7-C10 PFCAs were carried out.
These results can be effectively applied in biotechnologies of
environmental rehabilitation from contamination with
organofluorine compounds.
2. Materials and Methods
2.1. Bacterial Strain
Strain 5(3) was obtained from arable soil which had been affected
by pesticide contamination in the Yanaulsky district of the Republic
of Bashkortostan, Russian Federation (56.11381° North latitude,
54.766226° East longitude). The soil for isolating microorganisms
belonged to dark-grey forest soil (Haplic Greyzems) and was
characterized by the following parameters: total humus, 6.8%;
Ntotal, 0.60%; P2O5, 65.4 mg/kg; K2O, 112.5 mg/kg; available
copper and exchangeable zinc, 8.4 and 0.9 mg/kg soil, respectively;
and pH of water, 5.2. The pesticide contamination consisted of a
mixture including phosphorus-containing glyphosate (active
ingredient—N-(phosphonomethyl)glycine), chlorine-containing
octapone (active ingredient—2-(2,4-dichlorophenoxy)acetic acid),
and fluorine-containing fluroxalam (active ingredient—N-(2,6difluorophenyl)-8-fluoro-5-methoxy-[1,2,4]triazolo [1,5c]pyrimidine-2-sulfonamide). Strain 5(3) was deposited in the
collection of microorganisms of the Ufa Institute of Biology (IB

## Page 17

UFSC RAS) as UIB-251.
2.2. Chemicals and Reagents
The perfluorocarboxylic acids (PFCAs) in the following list were
used: perfluorodecanoic acid (PFDA), perfluorononic acid (PFNA),
perfluorooctanoic acid (PFOA), perfluoroheptanoic acid (PFHpA)
and perfluorooctanesulfonic acid (PFOS) (all high purity > 98%).
These were acquired from Sigma Aldrich (St. Louis, MO, USA).
Acetonitrile (quality for HPLC) was purchased from Merck
(Darmstadt, Germany). All remaining reagents were of analyticalgrade grade.
2.3. Genome Sequencing, Assembly and Annotation
The isolation of genomic DNA was conducted using a fresh culture
biomass (a colony) of strain 5(3), grown on LB agar using the
phenol-chloroform method according to the protocol [38].
Sequencing was performed on a GenoLab M system (GeneMind
Biosciences Co., Ltd., Luohu, China), and the library was prepared
with a ShotGun SG GM library preparation kit (cat. No SG_GM-96,
Raissol, Russia). A quality control of the reading was performed
using “HTQC” [38]. Low-quality (Q < 25) and short (<100 bp)
readings and adapter sequences were removed using Trimomatic
v.0.39 software [39]. Raw filtered readings were de novo assembled
using SPAdes version 3.15.4 software [40]. We employed Pilon
version 1.23 [41] and Bowtie2 version 2.3.5.1 [42] for error
correction purposes. In order to validate the circular nature of our
assembled replicon, we assessed the presence of overlapping ends.
To determine the strain’s species identity, we utilized the average
nucleotide identity (ANI) value (https://www.ezbiocloud.net/tools/
ani (accessed on 23 October 2023) [43]) and digital DNA-DNA
hybridization (DDH) analysis (https://ggdc.dsmz.de/ggdc.php
(accessed on 26 October 2023) [44]), with default settings.
We leveraged Prokka software v1.14.5 to annotate the genome
[45]. To unravel the functional potential of certain hypothetical
proteins, we employed the Basic Local Alignment Search Tool
(BLAST) [46].
This genome project has been deposited at GenBank under the
accession numbers JAUHUJ000000000 and under BioSample

## Page 18

number SAMN36271085, BioProject number PRJNA990579, and
SRA accession number SRR25905155.
We used FASTME 2.1.6.1 software [47] to generate a minimum
evolutionary tree using intergenomic distances. The tree was rooted
at the midpoint [48] and visualized through the online platform
https://itol.embl.de/ (accepted date, 30 October 2023) [49]. The
clustering of species and sub-species was performed following the
protocols described in references [50,51], respectively. To
determine the average nucleotide genome identity (ANI) between
strain 5(3) and its related strains, we employed the OrthoANI
algorithm [52].
2.4. Growing Environments and Conditions
Strain 5(3) was grown at 28 °C on Raymond liquid mineral medium
(gram per liter of distilled water): NH4NO3—2.0; MgSO4 × 7H2O—
0.2; KH2PO4—2.0; Na2HPO4—3; CaCl2 × 6H2O—0.01; and
Na2CO3—0.1 [53]. Additionally, PFDA, PFNA, PFOA, PFHpA or
PFOS (250 mg/L) was the sole carbon and energy source in an
orbital shaker–incubator ES-20/60 (SIA BIOSAN, Riga, Latvia) at
180 rpm.
To obtain the inoculate, the strain was grown on a Raymond
mineral medium with the addition of peptone (1 g/L) for 24 h.
Prior to inoculation into experimental flasks, the biomass
underwent precipitation and washing with sterile distilled water
and was added to experimental flasks until 0.1 OD600 was reached.
A lysogeny broth (LB) [54], consisting of (per liter of distilled
water) 10 g of trypton, 5 g of yeast extract, 5 g of NaCl and 15 g of
agar (PanReac, Barcelona, Spain), was used to isolate colonies and
determine the bacterial count.
In order to evaluate the efficiency of PFCA degradation, the strain
5(3) was cultivated at a temperature of 28 °C in a liquid mineral
medium containing individual C7-C10 PFCA compounds for 7 days.
The experimental protocols were carried out in three autonomous
biological repeats.
2.5. Isolation and Identification of PFCA Biotransformation
Products
The content of PFCA in the medium was evaluated, and the

## Page 19

products of their biotransformation were identified on a liquid
tandem chromatography-mass spectrometer LCMS-IT-TOF
(Shimadzu, Kyoto, Japan) (at the AGIDEL UFSC RAS Equipment
Collective Use Center) in ultrafiltrates (≤3 kDa) of culture liquids
obtained by ultrafiltration on Vivaflow 50 (Sartorius AG, Göttingen,
Germany), as described [21].
The Shim-pack XR-ODS column (75 mm × 2.0 mm id, 2.2 μm)
(Shimadzu, Kyoto, Japan) was used for UFLC analysis, in a gradient
elution system with 5 mM ammonium acetate in acetonitrile (A)
and 0.1% acetic acid (v/v) in water (B). A linear gradient elution
was employed, with the following specifications: from 0 min to 10
min, the composition transitioned from 60% to 30% (B); then, from
10 min to 20 min, the composition reversed, transitioning from
30% back to 60% (B). The chromatographic separation was
performed maintaining a steady flow rate of 0.25 mL/min,
employing an injection volume of 5 mL. Mass spectrometry data
were acquired from an electrospray ionization (ESI) source, which
was operated in negative ionization mode. MS operating conditions
were as follows: probe voltage, −3.5 kV; curve desolvation line
(CDL) and heat block temperature, 200 °C; detector voltage, 1.57 
kV; nebulizing gas (N2) flow, 1.5  L/min; collision gas and cooling
gas (argon), 50 kPa and 105 kPa, respectively. Parent ion m/z was
acquired in the 150–700 range, with an ion accumulation time of
100  ms (event time, 432  ms; repeat = 3). The timing and
separation characteristics of the chromatogram are shown in Figure
S1.
The structure of the obtained compounds was determined based on
a set of data from the analysis of mass spectra based on the decay of
a molecular ion and comparison with literature data.
Metabolic pathways were drawn using ChemSketch 2023.1.1.
The fluoride balance was determined by ion chromatography using
an LC-20 Prominence HPLC system with a CDD-10Avp
conductometric detector (Shimadzu, Japan). Fluorine ions were
separated on a Shodex column (Shodex, New York, NY, USA) at a
flow rate of 1 mL/min. An aqueous solution of sodium carbonate
and sodium bicarbonate was used as an eluent: 1.8 mM Na2CO3 +
1.7 mM NaHCO3. Post-column eluent suppression was achieved by
employing a Xenoic® XAMS suppressor coupled with an ASUREXA100 (Diduco AB, Umeå, Sweden).

## Page 20

2.6. Statistical Analysis
The statistical analysis was conducted utilizing Microsoft Office
Excel 2021 software.
3. Results and Discussion
3.1. Identification and Functional Annotation of the Genome of
Strain 5(3)
Cells of the strain 5(3) are Gram-negative, motile, with a single
polar flagellum, asporogenous and rod-shaped, able to produce a
fluorescent pigment during cultivation on a King B medium. On
nutrient agar, they form circular and non-pigmented colonies. The
growth temperature range lies in interval of 10–36 °C; the optimal
growth temperature is between 26 and 30 °C. The optimal pH value
is 6.8–8.0. They do not reduce nitrate to nitrite. Arginine
dihydrolase, catalase and cytochrome oxidase are produced. The
Voges–Proskauer reaction is negative.
The strain did not hydrolyze lecithin or starch but hydrolyzed
gelatin. The strain did not synthesize lipolytic enzymes and was not
able to grow on a medium with twin-80. At a concentration of 0–
5% NaCl, intensive growth was observed, and at a higher
concentration of up to 10% NaCl, weak growth was observed. The
strain 5(3) did not use xylose, rhamnose, galactose, malonate, Dtartrate, benzoate, or L-tryptophan as the sole source of carbon and
energy. Ribose, glucose, D-mannose, D-mannitol, ketogluconate, Lhistidine, L-rhamnose, histamine and D-sorbitol were used as
carbon sources.
Based on a preliminary analysis of physiological and biochemical
data and the 16S rRNA gene, the strain was identified as belonging
to the Pseudomonas genus. Pseudomonads have a large number of
assemblies represented in various databases such as NCBI, BV-BRS,
TYG, and Pseudomonas.com [55]. Detailed taxonomy of this genus
is still lacking and despite a long history of study and extensive
databases, it is constantly being supplemented with new findings.
As the result of the analysis of genome-wide sequencing, the strain
5(3) (Figure 1A) was reliably referred to as Pseudomonas mosselii,

## Page 21

and the values of the main identification parameters relative to the
reference (Pseudomonas mosselii DSM17497T) are as follows: ANI,
97.40%, and DDH, 80.30% (with species thresholds > 95% and
>70%, respectively [56]).
The sequencing and complete genome assembly of the strain
Pseudomonas mosselii 5(3) revealed that the assembly contained
5134 coding sequences (CDSs), four rRNA clusters, and 65 tRNAs.
The genome consists of 73 contigs with a total length of 5,676,241
bp. The N50 value is 165,927 bp, and the GC content is 64.38%.
Of the 5134 genes, 2895 (56.8%) were functionally annotated
(Figure 1B).
The result of the functional annotation shows that strain 5(3)
possesses genes of all metabolic pathways necessary for the
existence of an autonomous culture.
The category of “Xenobiotics biodegradation and metabolism” in
strain 5(3) contains more than 60 genes. Most of this category is
associated with the degradation of various aromatic compounds
(including ethylbenzene, nitrotoluene xylene, styrene, naphthalene,
polycyclic aromatic hydrocarbon, aminobenzoate, dioxin and
atrazine). According to [57], laccases and dehalogenases emerge as
the most promising contenders for PFCA biotransformation
enzymes. These enzymes are currently under the scrutiny of
scientists investigating biocatalysis as a tool for removing persistent
pollutants, including PFCA. Indeed, the putative enzymes
potentially involved in PFCA decarboxylation and decarbonylation
were found within the investigated genome with a 50% occurrence
rate. The genes that could participate in the destruction of
fluorinated organic compounds (dehalogenation) include the
haloalkane dehalogenase gene (dhaA) and the haloacetate
dehalogenase H-1 gene (dehH1). Like the previously characterized
fluoracetate dehalogenases [58], they can mediate the
defluorination of PFCA at the alpha-carbon position.
And the mechanisms of resistance to them can be mediated by the
presence of the following:
-
Decarboxylase gene (novR), which can carry out several
successive steps of the oxidative decarboxylation of PFCA;
this was previously described and shown, using a chemical
method, for similar compounds [59];

## Page 22

-
Alkanesulfonate monooxygenase gene (ssuE), catalyzing the
desulfonation of various organosulfonate substrates under
conditions of sulfate starvation, which can also produce a
protective role under conditions of oxidative stress [60];
-
Fluoride ion transporter gene (crcB). With an increase in
fluoride levels, it enhances the transcriptional activity of
genes located further downstream. It is assumes that these
genes aid in reducing the detrimental impacts caused by
excessive concentrations of fluoride [61], and assumes that
many genes are regulated by them. According to [62],
effective defluorination activity will require a high level of
tolerance to elevated intracellular concentrations of F− in
the microorganism.
Pseudomonas genomes possess a diverse array of enzymes with the
potential to facilitate defluorination reactions, utilizing both
oxygenase-mediated biodegradation processes and anaerobic
metabolic activities. While the complete functionality of these
enzymes in defluorination mechanisms remains undisclosed, it is
conceivable that these degradative enzymes might contribute to
Pseudomonas’ response to intracellular fluoride release.
In none of the previous publications devoted to the destruction of
organofluorine compounds by microorganisms have the authors
found any mention of the confirmed presence of the presented
complex of enzymes.
It should be remembered that Pseudomonas genomes encode >
25% of proteins with unknown function, which suggests that the
new defluorinating activity may belong to them [63,64].
3.2. Growth on C7-C10 PFCA and Their Defluorination
To grow on PFCA, bacteria need to transport it into the cell,
develop an efficient enzyme capable of catalyzing the cleavage of
the strong C-F bond, and detect a toxic fluorine ion and remove it
from the cell. And it seems that this was not a problem for the P.
mosselii strain 5(3). The strain actively grew on mineral media
containing C7-C10 PFCA as the only source of carbon (Figure 2A–E),
reaching the highest point of optical density of the culture liquid

## Page 23

after 6–7 days, and the maximum optical density was in the range
of 0.7–1.0 depending on the acid and was directly related to the
quantity of carbon atoms of the PFCA. With the growth of the strain
on C7-C10 perfluorocarboxylic acids, their complete decomposition
took place after 7 days, but 5 days were enough for the bacteria to
transform PFOS.
The results obtained in this study are significantly superior to those
presented earlier. The closest result to us was declared by the
authors in work [23] and was achieved using a strain of
Acidimicrobium sp. A6 for the removal of PFOA and PFOS in an
aerobic environment, where the degree of biodegradation did not
exceed 63% for a concentration of 100 mg/L for 100 days.
Only in contrast to them, we observed the release of F−, which is
an indicator of partial decomposition and perhaps these ions cause
the inhibition of the further destruction of intermediate fluorinated
compounds. The transformation of perfluorinated substrates was in
parallel with the release of free F− into the medium, and the
beginning of release correlated with the beginning of a linear
decrease in their concentration in the medium. The absence of an
inhibitory effect on bacterial growth is precisely explained by the
presence of the fluoride ion transporter gene, which allows the
bacteria to cross the physiological barrier of fluoride toxicity during
the decomposition of PFCAs, and to carry out defluorination,
regardless of the difference in redox potentials of the processes, at
the level of dechlorination of the polychlorinated compounds.
3.3. Biodegradation of C7-C10 PFCAs
The biological degradation of PFCAs may include several steps,
including decarboxylation, hydroxylation and defluorination
reactions.
During chromatographic analysis with mass spectrometry in the
initial culture liquid when cultivating strain 5(3) in a medium with
PFHpA, we observed its dissociated acid ions (molecular ion with
an m/z ratio of 363) (Figure 2F). A day later, a compound with m/z
319 was detected in the ultrafiltrate, which may occur as a result of
the elimination of the COO− group (m/z 44) from the carboxyl
group. Its concentration continued to increase up to 4 days, after
which a component with m/z 313, identified as perfluorohexanoic
acid (PFHxA), began to appear in the medium, presumably under

## Page 24

the action of the enzyme haloalkane dehalogenase. After 6 days,
this compound prevailed in the culture liquid. A similar series of
reactions took place during the conversion of homologous acids,
including PFOA (m/z 413) (Figure 2G), PFNA (m/z 463) (Figure
2H), and PFDA (m/z 513) (Figure 2I). Intermediate products
between cycles differed by the amount of mass of the cleaved CF2−
group, i.e., at m/z 50. Everywhere, PFHxA was found as the final
prevailing destruction product in an amount from 50 to 95% of the
total destruction products. It took two such reaction cycles to
decompose PFOA, three cycles to decompose PFNA, and four cycles
to decompose PFDA.
PFCA compounds, which also include C−S and C−C bonds, have
the potential for one-electron reduction or oxidation, followed by
hydroxylation.
Similarly, we assumed that sulfonate group in PFOS would first
break with the release of sulfite for bacterial metabolism, and that
the synthesis of perfluorooctane followed by the release of hydroxyl
and carboxyl groups and defluorination reactions results in the
formation of perfluoroheptanoic acid, as shown in our previous
work [21]. But in the realities of the experiment, everything was
different. During the decomposition of PFOS (m/z 499) (Figure 2J),
probably with the participation of haloacetate dehalogenase H-1,
free electrons attacked the most vulnerable spot, the α-position of
the fluorine of the alkyl chain, replacing two fluorine atoms with
hydrogen, followed by the elimination of the carbon–carbon bond
of the alkyl chain through an intermediate unidentified a product
with m/z 461, i.e., shortening the chain by CH2 with its release and
the formation of perfluoroheptane sulfonate (m/z 449) by day 2 of
cultivation. This was followed by a similar cycle and the formation
of perfluorohexane sulfonate (m/z 399) by day 4. And only after
that, under the probable action of alkanesulfonate monooxygenase,
the C−S bond was broken, and SO−3 was released to form a
compound with m/z 319, with its transformation into PFHxA, and
after 7 days, this compound was detected in an amount of at least
70% of the sum of the destruction products.
Thus, in the final stage of cultivation, the compounds detected in
the medium dissociated acid ion led to the identification of
perfluorohexanoic acid. The obtained data allowed us to construct
an assumed the scheme of destruction of C7-C10 PFCA (Figure 3).
Further investigation of the decomposition of substances by the

## Page 25

studied strain is currently not possible and requires further study of
its catabolic capabilities. From available sources, lower-molecularweight fragments produced by PFCA destruction were observed
only in [23] but without confirming the described genetic potential
and partial mineralization by fluorine release. The literature on the
toxicity of PFCA indicates that the shorter the F−C chain, the more
favorable its toxicological profile. Accordingly, there is a clear
rationale for reducing the toxicity of PFCA in relation to human
exposure, even by shortening its chain by one link [65,66,67].
3.4. Fluoride Balance
In our case, it could be assumed that the destruction stops as the
fluoride ion level in the medium reaches 19 mg/L (1 mM), the
value at which, according to [68,69], this anion becomes toxic at
intracellular levels. But in our early studies, microorganisms
remained viable at concentrations exceeding 150 mg/L [70].
Therefore, the suppression of PFCA defluorination by its
biodegradation products is more likely. And for cells initially having
the crcB gene, which encodes the fluoride exporting protein [71]
and controls the level of fluoride, when a potentially toxic
concentration of it occurs, they organize its displacement due to an
electrochemical gradient, which is shown by the example of P.
putida KT2440 [61].
According to the results obtained and the scheme presented, during
biodegradation, 26.3 (growth on PFHpA), 46.0 (on PFOA), 61.6 (on
PFNA), 74.1 (on PFDA), and 57.1 (on PFOS) mg/L fluoride ions
should have been released into the culture medium according to the
calculated data. In contrast to the expected value, the actually
detected concentration in all the studied variants did not exceed
24.4 mg/L, i.e., the degree of extraction of F- only in the case of
growth on PFHpA was stoichiometric and tended to 100%, but in
other variants, this value did not exceed 48.5%. Some authors
explain the non-stoichiometric release of fluorine during
biodegradation by its redirection by the cell for their needs. For
example, Pseudomonas sp. strain 273 is a destructor of
bichlorinated and fluorinated alkanes with medium and long chain
(C7-C16)—for phospholipid biosynthesis [72,73,74].

## Page 26

4. Conclusions
The Pseudomonas mosselii strain 5(3), which was isolated from
herbicide-contaminated soil, capable of destruction of C7-C10 PFCA,
was described for the first time. The genome of strain 5(3) is fully
sequenced, contains a total of 5134 genes—in particular, genes not
mentioned in microorganisms before—including the haloalkane
dehalogenase gene (dhaA), haloacetate dehalogenase H-1 gene
(dehH1), fluoride ion transporter gene (crcB) and alkanesulfonate
monooxygenase gene (ssuE), which are crucial for the degradation
of fluorinated compounds. The strain of P. mosselii 5(3), after 7
days of batch-cultivation with diverse of C7-C10 PFCAs as the sole
resource of carbon and energy, completely utilized them. The
results of the LC-MS analysis showed that the transformation takes
place due to perfluorohexanoic acid with the release of various
levels of stoichiometry (depending on PFCAs) of mineralization
indicators—fluorine ions determined by ion chromatography. Thus,
the Pseudomonas mosselii strain 5(3) demonstrates a genetically
confirmed high potential for the decomposition of C7-C10 PFCAs,
and the results obtained will help both in the fundamental
exploration of cellular processes of the strain and in the practice of
developing biotechnologies for environmental rehabilitation from
contamination with organofluorine compounds.
Supplementary Materials
The following supporting information can be downloaded at:
https://www.mdpi.com/article/10.3390/toxics11121001/s1, Figure
S1: Characteristics of LCMS distribution of C7-C10 PFCAs.
Author Contributions
Conceptualization, S.C.; methodology, S.C., S.S. and D.S.; software,
G.H.; validation, S.C., S.S. and D.C.; formal analysis, D.C.;
investigation, D.S., S.S. and D.C.; resources, S.C.; data curation, S.C.
and D.C.; writing—original draft preparation, S.C.; writing—review
and editing, S.C. and G.H.; visualization, G.H., D.S. and S.S.;

## Page 27

supervision, S.C.; project administration, S.C.; funding acquisition,
S.C. All authors have read and agreed to the published version of
the manuscript.
Funding
This research was funded by the Russian Science Foundation, grant
number 23-24-00154.
Institutional Review Board Statement
Not applicable.
Informed Consent Statement
Not applicable.
Data Availability Statement
The data that support the findings of this study are available from
the corresponding author on reasonable request.
Acknowledgments
The authors express their sincere gratitude to G.R. Kudoyarova for
correcting and editing the article in English.
Conflicts of Interest
The authors declare no conflict of interest. The funders had no role
in the design of the study; in the collection, analyses, or
interpretation of data; in the writing of the manuscript, or in the
decision to publish the results.

## Page 28

References
1. Ross, I.; McDonough, J.; Miles, J.; Storch, P. A review of
emerging technologies for remediation of PFASs. Remediation
2018, 28, 101–126. [Google Scholar] [CrossRef]
2. Report of the Conference of the Parties of the Stockholm
Convention on Persistent Organic Pollutants on the Work of
Its Fourth Meeting, 4–8 May//UNEP/POPS/COP.4/38;
Stockholm Convention Secretariat: Geneva, Switzerland,
2009; pp. 66–69.
3. Fujii, S.; Tanaka, S.; Hong Lien, N.P.; Qiu, Y.; Polprasert, C.
New POPs in the water environment: Distribution,
bioaccumulation and treatment of perfluorinated compounds:
A review paper. J. Water Supply Res. Technol.-AQUA 2007,
56, 313–326. [Google Scholar] [CrossRef]
4. Nascimento, R.A.; Nunoo, D.B.O.; Bizkarguenaga, E.; Schultes,
L.; Zabaleta, I.; Benskin, J.P.; Spano, S.; Leonel, J. Sulfluramid
use in Brazilian agriculture: A source of per- and
polyfluoroalkyl substances (PFASs) to the environment.
Environ. Pollut. 2018, 242, 1436–1443. [Google Scholar]
[CrossRef]
5. Xia, C.; Diamond, M.L.; Peaslee, G.F.; Peng, H.; Blum, A.;
Wang, Z.; Shalin, A.; Whitehead, H.D.; Green, M.; SchwartzNarbonne, H.; et al. Per- and polyfluoroalkyl substances in
North American school uniforms. Environ. Sci. Technol.
2022, 56, 13845–13857. [Google Scholar] [CrossRef]
6. Schaider, L.A.; Balan, S.A.; Blum, A.; Andrews, D.Q.; Strynar,
M.J.; Dickinson, M.E.; Lunderberg, D.M.; Lang, J.R.; Peaslee,
G.F. Fluorinated compounds in U.S. fast food packaging.
Environ. Sci. Technol. Lett. 2017, 4, 105–111. [Google
Scholar] [CrossRef]
7. Pozo, K.; Moriera, L.B.; Karaskova, P.; Pribylova, P.; Klanova,
J.; de Carvalho, M.U.; Maranho, L.A.; de Souza Abessa, D.M.
Using large amounts of firefighting foams releases per- and
polyfluoroalkyl substances (PFAS) into estuarine
environments: A baseline study in Latin America. Mar. Pollut.
Bull. 2022, 182, 113938. [Google Scholar] [CrossRef]
8. Prevedouros, K.; Cousins, I.T.; Buck, R.C.; Korzeniowski, S.H.
Sources, fate and transport of perfluorocarboxylates. Environ.

## Page 29

Sci. Technol. 2006, 40, 32–44. [Google Scholar] [CrossRef]
9. Wang, Y.; Munir, U.; Huang, Q. Occurrence of per- and
polyfluoroalkyl substances (PFAS) in soil: Sources, fate, and
remediation. Soil Environ. Health 2023, 1, 100004. [Google
Scholar] [CrossRef]
10. Ji, B.; Zhao, Y. World Profile of Foreseeable Strategies for the
Removal of Per- and Polyfluoroalkyl Substances (PFASs) from
Water. In Persistent Pollutants in Water and Advanced
Treatment Technology; Springer Nature: Singapore, 2023; pp.
47–69. [Google Scholar]
11. EPA 822-R-16-002; Health Effects Support Document for
Perfluorooctane Sulfonate (PFOS). U.S. Environmental
Protection Agency, Office of Water (4304T) Health and
Ecological Criteria Division: Washington, DC, USA, 2016.
12. EPA 822-R16-003; Health Effects Support Document for
Perfluorooctanoic Acid (PFOA). U.S. Environmental
Protection Agency, Office of Water (4304T) Health and
Ecological Criteria Division: Washington, DC, USA, 2016.
13. Seo, S.-H. Health risk of Human Exposure to Perfluorinated
Compounds (PFASs) in Hyeongsan River, Pohang. J. Environ.
Anal. Health Toxicol. 2022, 25, 77–84. [Google Scholar]
[CrossRef]
14. Calvert, L.; Green, M.P.; De Iuliis, G.N.; Dun, M.D.; Turner,
B.D.; Clarke, B.O.; Eamens, A.L.; Roman, S.D.; Nixon, B.
Assessment of the Emerging Threat Posed by Perfluoroalkyl
and Polyfluoroalkyl Substances to Male Reproduction in
Humans. Front. Endocrinol. 2022, 12, 799043. [Google
Scholar] [CrossRef]
15. Cheng, J.; Vecitis, C.D.; Park, H.; Mader, B.T.; Hoffmann,
M.R. Sonochemical degradation of perfluorooctane sulfonate
(PFOS) and perfluorooctanoate (PFOA) in landfill
groundwater: Environmental matrix effects. Environ. Sci.
Technol. 2008, 42, 8057–8063. [Google Scholar] [CrossRef]
16. Wang, F.; Shih, K.; Lu, X.; Liu, C. Mineralization behavior of
fluorine in perfluorooctanesulfonate (PFOS) during thermal
treatment of lime-conditioned sludge. Environ. Sci. Technol.
2013, 47, 2621–2627. [Google Scholar] [CrossRef]
17. Marquínez-Marquínez, A.N.; Loor-Molina, N.S.; QuirozFernández, L.S.; Maddela, N.R.; Luque, R.; Rodríguez-Díaz,
J.M. Recent advances in the remediation of

## Page 30

perfluoroalkylated and polyfluoroalkylated contaminated
sites. Environ. Res. 2023, 219, 115152. [Google Scholar]
[CrossRef]
18. Kabiri, S.; Navarro, D.A.; Hamad, S.A.; Grimison, C.; Higgins,
C.P.; Mueller, J.F.; Kookana, R.S.; McLaughlin, M.J. Physical
and chemical properties of carbon-based sorbents that affect
the removal of per- and polyfluoroalkyl substances from
solution and soil. Sci. Total Environ. 2023, 875, 162653.
[Google Scholar] [CrossRef]
19. Mayer-Blackwell, K.; Sewell, H.; Fincker, M.; Spormann, A.M.
Comparative physiology of organohalide-respiring bacteria. In
Organohalide-Respiring Bacteria; Adrian, L., Löffler, F.E.,
Eds.; Springer: Berlin/Heidelberg, Germany, 2016; pp. 259–
280. [Google Scholar]
20. Wackett, L.P. Why Is the biodegradation of polyfluorinated
compounds so rare? mSphere 2021, 6, 10–1128. [Google
Scholar] [CrossRef]
21. Chetverikov, S.P.; Sharipov, D.A.; Korshunova, T.Y.; Loginov,
O.N. Degradation of perfluorooctanyl sulfonate by strain
Pseudomonas plecoglossicida 2.4-D. Appl. Biochem.
Microbiol. 2017, 53, 533–538. [Google Scholar] [CrossRef]
22. Shaw, D.M.J.; Munoz, G.; Bottos, E.M.; Duy, S.V.; Sauve, S.;
Liu, J.; Van Hamme, J.D. Degradation and defluorination of
6:2 fluorotelomer sulfonamidoalkyl betaine and 6:2
fluorotelomer sulfonate by Gordonia sp. strain NB4-1Y under
sulfur-limiting conditions. Sci. Total Environ. 2019, 647,
690–698. [Google Scholar] [CrossRef]
23. Huang, S.; Jaffé, P.R. Defluorination of perfluorooctanoic acid
(PFOA) and perfluorooctane sulfonate (PFOS) by
Acidimicrobium sp. Strain A6. Environ. Sci. Technol. 2019,
53, 11410–11419. [Google Scholar] [CrossRef]
24. Yu, Y.; Zhang, K.; Li, Z.; Ren, C.; Chen, J.; Lin, Y.-H.; Liu, J.;
Men, Y. Microbial cleavage of c–f bonds in two C6 Per- and
polyfluorinated compounds via reductive defluorination.
Environ. Sci. Technol. 2020, 54, 14393–14402. [Google
Scholar] [CrossRef]
25. Ruiz-Urigüen, M.; Shuai, W.; Huang, S.; Jaffé, P.R.
Biodegradation of PFOA in microbial electrolysis cells by
Acidimicrobiaceae sp. strain A6. Chemosphere 2022, 292,
133506. [Google Scholar] [CrossRef]

## Page 31

26. Kwon, B.G.; Lim, H.-J.; Na, S.-H.; Choi, B.-I.; Shin, D.-S.;
Chung, S.-Y. Biodegradation of perfluorooctanesulfonate
(PFOS) as an emerging contaminant. Chemosphere 2014,
109, 221–225. [Google Scholar] [CrossRef]
27. Yi, L.B.; Chai, L.Y.; Xie, Y.; Peng, Q.J.; Peng, Q.Z. Isolation,
identification, and degradation performance of a PFOAdegrading strain. Genet. Mol. Res. 2016, 15, 235–246.
[Google Scholar] [CrossRef]
28. Beškoski, V.P.; Yamamoto, A.; Nakano, T.; Yamamoto, K.;
Matsumura, C.; Motegi, M.; Beškoski, L.S.; Inui, H.
Defluorination of perfluoroalkyl acids is followed by
production of monofluorinated fatty acids. Sci. Total Environ.
2018, 15, 355–359. [Google Scholar] [CrossRef]
29. Chiriac, F.L.; Stoica, C.; Iftode, C.; Pirvu, F.; Petre, V.A.; Paun,
I.; Pascu, L.F.; Vasile, G.G.; Nita-Lazar, M. Bacterial
Biodegradation of Perfluorooctanoic Acid (PFOA) and
Perfluorosulfonic Acid (PFOS) Using Pure Pseudomonas
Strains. Sustainability 2023, 15, 14000. [Google Scholar]
[CrossRef]
30. Zhang, Z.; Sarkar, D.; Biswas, J.K.; Datta, R. Biodegradation
of per- and polyfluoroalkyl substances (PFAS): A review.
Bioresour. Technol. 2022, 344, 126223. [Google Scholar]
[CrossRef]
31. Kiel, M.; Karl-Heinrich, E. The biodegradation vs.
biotransformation of fluorosubstituted aromatics. Appl.
Microbiol. Biotechnol. 2015, 99, 7433–7464. [Google
Scholar] [CrossRef]
32. Nogales, J.; García, J.L.; Díaz, E. Degradation of Aromatic
Compounds in Pseudomonas: A Systems Biology View. In
Aerobic Utilization of Hydrocarbons, Oils and Lipids:
Handbook of Hydrocarbon and Lipid Microbiology; Rojo, F.,
Ed.; Springer: Cham, Switzerland, 2017; pp. 1–49. [Google
Scholar]
33. Yi, L.; Tang, C.; Peng, Q.; Peng, Q.; Chai, L. Draft genome
sequence of perfluorooctane acid-degrading bacterium
Pseudomonas parafulva YAB-1. Genome Announc. 2015, 3,
e00935-15. [Google Scholar] [CrossRef]
34. Harris, J.; Gross, M.; Kemball, J.; Farajollahi, S.; Dennis, P.;
Sitko, J.; Steel, J.J.; Almand, E.; Kelley-Loughnane, N.;
Varaljay, V.A. Draft Genome Sequence of the Bacterium

## Page 32

Delftia acidovorans Strain D4B, Isolated from Soil. Microbiol.
Resour. Announc. 2021, 10, e0063521. [Google Scholar]
[CrossRef]
35. Spaan, K.M.; van Noordenburg, C.; Plassmann, M.M.;
Schultes, L.; Shaw, S.; Berger, M.; Heide-Jørgensen, M.P.;
Rosing-Asvid, A.; Granquist, S.M.; Dietz, R.; et al. Fluorine
mass balance and suspect screening in marine mammals from
the northern hemisphere. Environ. Sci. Technol. 2020, 54,
4046–4058. [Google Scholar] [CrossRef]
36. Brendel, S.; Fetter, É.; Staude, C.; Vierke, L.; Biegel-Engler, A.
Short-chain perfluoroalkyl acids: Environmental concerns and
a regulatory strategy under REACH. Environ. Sci. Eur. 2018,
30, 9. [Google Scholar] [CrossRef]
37. Li, F.; Duan, J.; Tian, S.; Ji, H.; Zhu, Y.; Wei, Z.; Zhao, D.
Short-chain per- and polyfluoroalkyl substances in aquatic
systems: Occurrence, impacts and treatment. Chem. Eng. J.
2020, 380, 122506. [Google Scholar] [CrossRef]
38. Sambrook, J.; Russell, D.W. Purification of nucleic acids by
extraction with phenol: Chloroform. Cold Spring Harb.
Protoc. 2006, 2006, pdb.prot4455. [Google Scholar]
39. Bolger, A.M.; Lohse, M.; Usadel, B. Trimmomatic: A flexible
trimmer for Illumina Sequence Data. Bioinformatics 2014, 30,
2114–2120. [Google Scholar] [CrossRef]
40. Bankevich, A.; Nurk, S.; Antipov, D.; Gurevich, A.; Dvorkin,
M.; Kulikov, A.S.; Lesin, V.; Nikolenko, S.; Pham, S.;
Prjibelski, A.; et al. SPAdes: A new genome assembly
algorithm and its applications to single-cell sequencing. J.
Comput. Biol. 2012, 19, 455–477. [Google Scholar]
[CrossRef]
41. Langmead, B.; Salzberg, S.L. Fast gapped-read alignment with
Bowtie 2. Nat. Methods 2012, 9, 357–359. [Google Scholar]
[CrossRef]
42. Walker, B.J.; Abeel, T.; Shea, T.; Priest, M.; Abouelliel, A.;
Sakthikumar, S.; Cuomo, C.A.; Zeng, Q.; Wortman, J.; Young,
S.K. Pilon: An integrated tool for comprehensive microbial
variant detection and genome assembly improvement. PLoS
ONE 2014, 9, e112963. [Google Scholar] [CrossRef]
43. Yoon, S.H.; Ha, S.M.; Lim, J.M.; Kwon, S.J.; Chun, J. A largescale evaluation of algorithms to calculate average nucleotide
identity. Antonie Van Leeuwenhoek 2017, 110, 1281–1286.

## Page 33

[Google Scholar] [CrossRef]
44. Meier-Kolthoff, J.P.; Sardà Carbasse, J.; Peinado-Olarte, R.L.;
Göker, M. TYGS and LPSN: A database tandem for fast and
reliable genome-based classification and nomenclature of
prokaryotes. Nucleic Acid Res. 2022, 50, D801–D807.
[Google Scholar] [CrossRef]
45. Seemann, T. Prokka: Rapid prokaryotic genome annotation.
Bioinformatics 2014, 30, 2068–2069. [Google Scholar]
[CrossRef]
46. Altschul, S.F.; Gish, W.; Miller, W.; Myers, E.W.; Lipman, D.J.
Basic local alignment search tool. J. Mol. Biol. 1990, 215,
403–410. [Google Scholar] [CrossRef]
47. Lefort, V.; Desper, R.; Gascuel, O. FastME 2.0: A
Comprehensive, Accurate, and Fast Distance-Based Phylogeny
Inference Program. Mol. Biol. Evol. 2015, 32, 2798–2800.
[Google Scholar] [CrossRef]
48. Farris, J.S. Estimating phylogenetic trees from distance
matrices. Am. Nat. 1972, 6, 645–667. [Google Scholar]
[CrossRef]
49. Letunic, I.; Bork, P. Interactive Tree of Life (iTOL) v5: An
online tool for phylogenetic tree display and annotation.
Nucleic Acids Res. 2021, 49, W293–W296. [Google Scholar]
[CrossRef]
50. Meier-Kolthoff, J.P.; Goker, M. TYGS is an automated highthroughput platform for state-of-the-art genome-based
taxonomy. Nat. Commun. 2019, 10, 2182. [Google Scholar]
[CrossRef]
51. Meier-Kolthoff, J.P.; Hahnke, R.L.; Petersen, J.; Scheuner, C.;
Michael, V.; Fiebig, A.; Rohde, C.; Rohde, M.; Fartmann, B.;
Goodwin, L.A.; et al. Complete genome sequence of DSM
30083(T), the type strain (U5/41(T)) of Escherichia coli, and
a proposal for delineating subspecies in microbial taxonomy.
Stand. Genom. Sci. 2014, 9, 2. [Google Scholar] [CrossRef]
52. Lee, I.; Ouk Kim, Y.; Park, S.C.; Chun, J. OrthoANI: An
improved algorithm and software for calculating average
nucleotide identity. Int. J. Syst. Evol. Microbiol. 2016, 66,
1100–1103. [Google Scholar] [CrossRef]
53. Raymond, R.L. Microbial oxidation of n-paraffinic
hydrocarbons. Dev. Ind. Microbiol. 1961, 2, 23–54. [Google
Scholar]

## Page 34

54. Bertani, G. Studies on Lysogenesis I. J. Bacteriol. 1951, 62,
293–300. [Google Scholar] [CrossRef]
55. Olson, R.D.; Assaf, R.; Brettin, T.; Conrad, N.; Cucinell, C.;
Davis, J.J.; Dempsey, D.M.; Dickerman, A.; Dietrich, E.M.;
Kenyon, R.W.; et al. Introducing the Bacterial and Viral
Bioinformatics Resource Center (BV-BRC): A resource
combining PATRIC, IRD and ViPR. Nucleic Acids Res. 2023,
51, D678–D689. [Google Scholar] [CrossRef]
56. Chun, J.; Oren, A.; Ventosa, A.; Christensen, H.; Arahal, D.R.;
da Costa, M.S.; Rooney, A.P.; Yi, H.; Xu, X.W.; De Meyer, S.;
et al. Proposed minimal standards for the use of genome data
for the taxonomy of prokaryotes. Int. J. Syst. Evol. Microbiol.
2018, 68, 461–466. [Google Scholar] [CrossRef]
57. Marchetto, F.; Roverso, M.; Righetti, D.; Bogialli, S.; Filippini,
F.; Bergantino, E.; Sforza, E. Bioremediation of Per- and PolyFluoroalkyl Substances (PFAS) by Synechocystis sp. PCC
6803: A Chassis for a Synthetic Biology Approach. Life 2021,
11, 1300. [Google Scholar] [CrossRef] [PubMed]
58. Li, Y.; Yue, Y.; Zhang, H.; Yang, Z.; Wang, H.; Tian, S.; Wang,
J.; Zhang, Q.; Wang, W. Harnessing Fluoroacetate
Dehalogenase for Defluorination of Fluorocarboxylic Acids: In
Silico and In Vitro. Approach. Environ. Int. 2019, 131,
104999. [Google Scholar] [CrossRef]
59. Trang, B.; Li, Y.; Xue, X.S.; Ateia, M.; Houk, K.N.; Dichtel,
W.R. Low-temperature mineralization of perfluorocarboxylic
acids. Science 2022, 377, 839–845. [Google Scholar]
[CrossRef] [PubMed]
60. Park, C.; Shin, B.; Park, W. Protective Role of Bacterial
Alkanesulfonate Monooxygenase under Oxidative Stress.
Appl. Environ. Microbiol. 2020, 86, e00692-20. [Google
Scholar] [CrossRef] [PubMed]
61. Calero, P.; Gurdo, N.; Nikel, P.I. Role of the CrcB transporter
of Pseudomonas putida in the multi-level stress response
elicited by mineral fluoride. Environ. Microbiol. 2022, 24,
5082–5104. [Google Scholar] [CrossRef]
62. Wackett, L.P. Pseudomonas: Versatile biocatalysts for PFAS.
Environ. Microbiol. 2022, 24, 2882–2889. [Google Scholar]
[CrossRef]
63. Winsor, G.L.; Van Rossum, T.; Lo, R.; Khaira, B.; Whiteside,
M.D.; Hancock, R.E.; Brinkman, F.S. Pseudomonas Genome

## Page 35

Database: Facilitating user-friendly, comprehensive
comparisons of microbial genomes. Nucleic Acids Res. 2009,
37, D483–D488. [Google Scholar] [CrossRef]
64. Winsor, G.L.; Griffiths, E.J.; Lo, R.; Dhillon, B.K.; Shay, J.A.;
Brinkman, F.S. Enhanced annotations and features for
comparing thousands of Pseudomonas genomes in the
Pseudomonas genome database. Nucleic Acids Res. 2016, 44,
D646–D653. [Google Scholar] [CrossRef] [PubMed]
65. Luz, A.L.; Anderson, J.K.; Goodrum, P.; Durda, J.
Perfluorohexanoic acid toxicity, part I: Development of a
chronic human health toxicity value for use in risk
assessment. Regul. Toxicol. Pharmacol. 2019, 103, 41–55.
[Google Scholar] [CrossRef] [PubMed]
66. Zango, Z.U.; Ethiraj, B.; Al-Mubaddel, F.S.; Alam, M.M.;
Lawal, M.A.; Kadir, H.A.; Khoo, K.S.; Garba, Z.N.; Usman, F.;
Zango, M.U.; et al. An overview on human exposure, toxicity,
solid-phase microextraction and adsorptive removal of
perfluoroalkyl carboxylic acids (PFCAs) from water matrices.
Environ. Res. 2023, 231 Pt 2, 116102. [Google Scholar]
[CrossRef]
67. Han, J.S.; Jang, S.; Son, H.Y.; Kim, Y.B.; Kim, Y.; Noh, J.H.;
Kim, M.J.; Lee, B.S. Subacute dermal toxicity of
perfluoroalkyl carboxylic acids: Comparison with different
carbon-chain lengths in human skin equivalents and systemic
effects of perfluoroheptanoic acid in Sprague Dawley rats.
Arch. Toxicol. 2020, 94, 523–539. [Google Scholar]
[CrossRef]
68. Ji, C.; Stockbridge, R.B.; Miller, C. Bacterial fluoride
resistance, Fluc channels and the weak acid accumulation
effect. J. Gen. Physiol. 2014, 144, 257–261. [Google Scholar]
[CrossRef]
69. McIlwain, B.C.; Michal, T.R.; Stockbridge, R.B. Membrane
exporters of fluoride ion. Ann. Rev. Biochem. 2021, 90, 559–
579. [Google Scholar] [CrossRef] [PubMed]
70. Chetverikov, S.P.; Loginov, O.N. A new Ensifer adhaerens
strain M1 is capable of transformation of perfluorocarboxylic
acids. Microbiology 2019, 88, 115–117. [Google Scholar]
[CrossRef]
71. Stockbridge, R.B.; Lim, H.H.; Otten, R.; Williams, C.; Shane,
T.; Weinberg, Z.; Miller, C. Fluoride resistance and transport

## Page 36

by riboswitch-controlled CLC antiporters. Proc. Natl. Acad.
Sci. USA 2012, 109, 15289–15294. [Google Scholar]
[CrossRef]
72. Xie, Y.; Chen, G.; May, A.L.; Yan, J.; Brown, L.P.; Powers,
J.B.; Campagna, S.R.; Löffler, F.E. Pseudomonas sp. strain 273
degrades fluorinated alkanes. Environ. Sci. Technol. 2020,
54, 14994–15003. [Google Scholar] [CrossRef] [PubMed]
73. Xie, Y.; May, A.L.; Chen, G.; Brown, L.P.; Powers, J.B.; Tague,
E.D.; Campagna, S.R.; Löffler, F.E. Pseudomonas sp. strain
273 incorporates organofluorine into the lipid bilayer during
growth with fluorinated alkanes. Environ. Sci. Technol. 2022,
56, 8155–8166. [Google Scholar] [CrossRef] [PubMed]
74. Xie, Y.; Chen, G.; Ramirez, D.; Yan, J.; Löffler, F.E. Complete
Genome Sequence of Pseudomonas sp. Strain 273, a
Haloalkane-Degrading Bacterium. Microbiol. Resour.
Announc. 2023, 12, e0017623. [Google Scholar] [CrossRef]
Figure 1. Whole-genome tree of the position of strain 5(3) within
the genus Pseudomonas (A) and the number of genes associated
with common functional categories in its genome according to the
KEGG classification (B).
Figure 1. Whole-genome tree of the position of strain 5(3) within
the genus Pseudomonas (A) and the number of genes associated
with common functional categories in its genome according to the
KEGG classification (B).
Figure 2. Dynamics of growth of Pseudomonas mosselii strain 5(3)
(OD600), release of fluorine ions, changes in PFCA concentration
(graphs on the left) and percentages of its and its degradation
products (according to m/z) (histograms on the right) during
cultivation in a liquid mineral medium: (A,F)—PFHpA; (B,G)—
PFOA; (C,H)—PFNA; (D,I)—PFDA; (E,J)—PFOS.
Figure 2. Dynamics of growth of Pseudomonas mosselii strain 5(3)
(OD600), release of fluorine ions, changes in PFCA concentration
(graphs on the left) and percentages of its and its degradation
products (according to m/z) (histograms on the right) during
cultivation in a liquid mineral medium: (A,F)—PFHpA; (B,G)—
PFOA; (C,H)—PFNA; (D,I)—PFDA; (E,J)—PFOS.
Figure 3. The proposed C7-C10 PFCA biodegradation pathways of
Pseudomonas mosselii 5(3) on the basis of metabolite analysis by

## Page 37

LC-MS, ion chromatography and the results of whole genome
sequencing.
Figure 3. The proposed C7-C10 PFCA biodegradation pathways of
Pseudomonas mosselii 5(3) on the basis of metabolite analysis by
LC-MS, ion chromatography and the results of whole genome
sequencing.
Disclaimer/Publisher’s Note: The statements, opinions and data
contained in all publications are solely those of the individual
author(s) and contributor(s) and not of MDPI and/or the editor(s).
MDPI and/or the editor(s) disclaim responsibility for any injury to
people or property resulting from any ideas, methods, instructions
or products referred to in the content.
© 2023 by the authors. Licensee MDPI, Basel, Switzerland. This
article is an open access article distributed under the terms and
conditions of the Creative Commons Attribution (CC BY) license
(https://creativecommons.org/licenses/by/4.0/).
Share and Cite
MDPI and ACS Style
Chetverikov, S.; Hkudaygulov, G.; Sharipov, D.; Starikov, S.;
Chetverikova, D. Biodegradation Potential of C7-C10
Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics 2023, 11, 1001. https://
doi.org/10.3390/toxics11121001
AMA Style
Chetverikov S, Hkudaygulov G, Sharipov D, Starikov S,
Chetverikova D. Biodegradation Potential of C7-C10
Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics. 2023; 11(12):1001.
https://doi.org/10.3390/toxics11121001
Chicago/Turabian Style
Chetverikov, Sergey, Gaisar Hkudaygulov, Danil Sharipov, Sergey

## Page 38

Starikov, and Darya Chetverikova. 2023. "Biodegradation Potential
of C7-C10 Perfluorocarboxylic Acids and Data from the Genome of a
New Strain of Pseudomonas mosselii 5(3)" Toxics 11, no. 12: 1001.
https://doi.org/10.3390/toxics11121001
APA Style
Chetverikov, S., Hkudaygulov, G., Sharipov, D., Starikov, S., &
Chetverikova, D. (2023). Biodegradation Potential of C7-C10
Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics, 11(12), 1001. https://
doi.org/10.3390/toxics11121001
Note that from the first issue of 2016, this journal uses article
numbers instead of page numbers. See further details here.
Article Metrics
Supplementary Material
Supplementary File 1:
ZIP-Document (ZIP, 164 KB)
clear
Cite
Export citation file: BibTeX | EndNote | RIS
MDPI and ACS Style
Chetverikov, S.; Hkudaygulov, G.; Sharipov, D.; Starikov, S.;
Chetverikova, D. Biodegradation Potential of C7-C10
Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics 2023, 11, 1001. https://
doi.org/10.3390/toxics11121001
AMA Style
Chetverikov S, Hkudaygulov G, Sharipov D, Starikov S,
Chetverikova D. Biodegradation Potential of C7-C10

## Page 39

Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics. 2023; 11(12):1001.
https://doi.org/10.3390/toxics11121001
Chicago/Turabian Style
Chetverikov, Sergey, Gaisar Hkudaygulov, Danil Sharipov, Sergey
Starikov, and Darya Chetverikova. 2023. "Biodegradation Potential
of C7-C10 Perfluorocarboxylic Acids and Data from the Genome of a
New Strain of Pseudomonas mosselii 5(3)" Toxics 11, no. 12: 1001.
https://doi.org/10.3390/toxics11121001
APA Style
Chetverikov, S., Hkudaygulov, G., Sharipov, D., Starikov, S., &
Chetverikova, D. (2023). Biodegradation Potential of C7-C10
Perfluorocarboxylic Acids and Data from the Genome of a New
Strain of Pseudomonas mosselii 5(3). Toxics, 11(12), 1001. https://
doi.org/10.3390/toxics11121001
Note that from the first issue of 2016, this journal uses article
numbers instead of page numbers. See further details here.
clear
Toxics, EISSN 2305-6304, Published by MDPI
RSS Content Alert
Further Information
Article Processing Charges Pay an Invoice Open Access Policy
Contact MDPI Jobs at MDPI
Guidelines
For Authors For Reviewers For Editors For Librarians For Publishers
For Societies For Conference Organizers
MDPI Initiatives
Sciforum MDPI Books Preprints.org Scilit SciProfiles Encyclopedia
JAMS Proceedings Series

## Page 40

Follow MDPI
LinkedIn Facebook X
[image] Accounting and Auditing Acoustics Acta Microbiologica
Hellenica Actuators Adhesives Administrative Sciences Adolescents
Advances in Respiratory Medicine Aerobiology Aerospace
Agriculture AgriEngineering Agrochemicals Agronomy AI AI
Chemistry AI in Education AI Materials AI Sensors Air Algorithms
Allergies Alloys Analytica Analytics Anatomia Anesthesia Research
Animals Antibiotics Antibodies Antioxidants Applied Biosciences
Applied Mechanics Applied Microbiology Applied Nano Applied
Sciences Applied System Innovation AppliedChem AppliedMath
AppliedPhys Aquaculture Journal Architecture Arthropoda Arts
Astronautics Astronomy Atmosphere Atoms Audiology Research
Automation Axioms Bacteria Batteries Behavioral Sciences
Beverages Big Data and Cognitive Computing BioChem
Bioengineering Biologics Biology Biology and Life Sciences Forum
Biomass Biomechanics BioMed Biomedicines BioMedInformatics
Biomimetics Biomolecules Biophysica Bioresources and Bioproducts
Biosensors Biosphere BioTech Birds Blockchains Brain Sciences
Buildings Businesses C Cancers Cardiogenetics Cardiovascular
Medicine Catalysts Cells Ceramics Challenges ChemEngineering
Chemistry Chemistry Proceedings Chemosensors Children Chips
CivilEng Clean Technologies Climate Clinical and Translational
Neuroscience Clinical Bioenergetics Clinics and Practice Clocks &
Sleep Coasts Coatings Colloids and Interfaces Colorants
Commodities Complexities Complications Compounds Computation
Computer Sciences & Mathematics Forum Computers Condensed
Matter Conservation Construction Materials Corrosion and Materials
Degradation Cosmetics COVID Craniomaxillofacial Trauma &
Reconstruction Crops Cryo Cryptography Crystals Culture Current
Issues in Molecular Biology Current Oncology Dairy Data Dentistry
Journal Dermato Dermatopathology Designs Diabetology
Diagnostics Dietetics Digital Disabilities Diseases Diversity DNA
Drones Drugs and Drug Candidates Dynamics Earth Ecologies
Econometrics Economies Education Sciences Electricity Electrochem
Electronic Materials Electronics Emergency Care and Medicine
Encyclopedia Endocrines Energies Energy Storage and Applications
Eng Engineering Proceedings Entropic and Disordered Matter
Entropy Environmental and Earth Sciences Proceedings

## Page 41

Environments Epidemiologia Epigenomes European Burn Journal
European Journal of Investigation in Health, Psychology and
Education Family Sciences Fermentation Fibers FinTech Fire Fishes
Fluids Foods Forecasting Forensic Sciences Forests Fossil Studies
Foundations Fractal and Fractional Fuels Future Future Internet
Future Pharmacology Future Transportation Galaxies Games Gases
Gastroenterology Insights Gastrointestinal Disorders Gastronomy
Gels Genealogy Genes Geographies GeoHazards Geomatics
Geometry Geosciences Geotechnics Geriatrics GERMS Glacies Gout,
Urate, and Crystal Deposition Disease Grasses Green Health
Hardware Healthcare Hearts Hemato Hematology Reports Heritage
Histories Horticulturae Hospitals Humanities Humans Hydrobiology
Hydrogen Hydrology Hydropower Hygiene Immuno Infectious
Disease Reports Informatics Information Infrastructures Inorganics
Insects Instruments Intelligent Infrastructure and Construction
International Journal of Cognitive Sciences International Journal of
Environmental Medicine International Journal of Environmental
Research and Public Health International Journal of Financial
Studies International Journal of Molecular Sciences International
Journal of Neonatal Screening International Journal of Orofacial
Myology and Myofunctional Therapy International Journal of Plant
Biology International Journal of Topology International Journal of
Translational Medicine International Journal of Turbomachinery,
Propulsion and Power International Medical Education Inventions
IoT ISPRS International Journal of Geo-Information J Journal of
Aesthetic Medicine Journal of Ageing and Longevity Journal of
CardioRenal Medicine Journal of Cardiovascular Development and
Disease Journal of Clinical & Translational Ophthalmology Journal
of Clinical Medicine Journal of Composites Science Journal of
Cybersecurity and Privacy Journal of Dementia and Alzheimer's
Disease Journal of Developmental Biology Journal of Experimental
and Theoretical Analyses Journal of Eye Movement Research
Journal of Functional Biomaterials Journal of Functional
Morphology and Kinesiology Journal of Fungi Journal of Imaging
Journal of Intelligence Journal of Low Power Electronics and
Applications Journal of Manufacturing and Materials Processing
Journal of Marine Science and Engineering Journal of Market
Access & Health Policy Journal of Mind and Medical Sciences
Journal of Molecular Pathology Journal of Nanotheranostics
Journal of Nuclear Engineering Journal of Otorhinolaryngology,

## Page 42

Hearing and Balance Medicine Journal of Parks Journal of
Personalized Medicine Journal of Pharmaceutical and BioTech
Industry Journal of Respiration Journal of Risk and Financial
Management Journal of Sensor and Actuator Networks Journal of
the Oman Medical Association Journal of Theoretical and Applied
Electronic Commerce Research Journal of Vascular Diseases Journal
of Xenobiotics Journal of Zoological and Botanical Gardens
Journalism and Media Kidney and Dialysis Kinases and
Phosphatases Knowledge LabMed Laboratories Land Languages
Laws Life Lights Limnological Review Lipidology Liquids Literature
Livers Logics Logistics Lubricants Lymphatics Machine Learning and
Knowledge Extraction Machines Macromol Magnetism
Magnetochemistry Marine Drugs Materials Materials Proceedings
Mathematical and Computational Applications Mathematics
Medical Sciences Medical Sciences Forum Medicina Medicines
Membranes Merits Metabolites Metals Meteorology Methane
Methods and Protocols Metrics Metrology Micro Microbiology
Research Microelectronics Micromachines Microorganisms
Microplastics Microwave Minerals Mining Modelling Modern
Mathematical Physics Molbank Molecules Multimedia Multimodal
Technologies and Interaction Muscles Nanoenergy Advances
Nanomanufacturing Nanomaterials NDT Network Neuroglia
Neurology International NeuroSci Nitrogen Non-Coding RNA
Nursing Reports Nutraceuticals Nutrients Obesities Occupational
Health Oceans Onco Optics Oral Organics Organoids Osteology
Oxygen Parasitologia Particles Pathogens Pathophysiology Peace
Studies Pediatric Reports Pets Pharmaceuticals Pharmaceutics
Pharmacoepidemiology Pharmacy Philosophies Photochem
Photonics Phycology Physchem Physical Sciences Forum Physics
Physiologia Plants Plasma Platforms Pollutants Polymers
Polysaccharides Populations Poultry Powders Precision Oncology
Proceedings Processes Prosthesis Proteomes Psychiatry International
Psychoactives Psychology International Publications Purification
Quantum Beam Science Quantum Reports Quaternary Radiation
Reactions Real Estate Receptors Recycling Regional Science and
Environmental Economics Religions Remote Sensing Reports
Reproductive Medicine Resources Rheumato Risks Robotics
Romanian Journal of Preventive Medicine Ruminants Safety Sci
Scientia Pharmaceutica Sclerosis Seeds Sensors Separations Sexes
Signals Sinusitis Smart Cities Social Sciences Société Internationale

## Page 43

d’Urologie Journal Societies Software Soil Systems Solar Solids
Spectroscopy Journal Sports Standards Stats Stresses Surfaces
Surgeries Surgical Techniques Development Sustainability
Sustainable Chemistry Symmetry SynBio Systems Targets Taxonomy
Technologies Telecom Textiles Thalassemia Reports Theoretical and
Applied Ergonomics Therapeutics Thermo Time and Space
Tomography Tourism and Hospitality Toxics Toxins
Transplantology Trauma Care Trends in Higher Education Tropical
Medicine and Infectious Disease Universe Urban Science Uro
Vaccines Vehicles Venereology Veterinary Sciences Vibration
Virtual Worlds Viruses Vision Waste Water Wild Wind Women
World World Electric Vehicle Journal Youth Zoonotic Diseases
Subscribe
Subscribe to receive issue release notifications and newsletters from
MDPI journals
© 1996-2025 MDPI (Basel, Switzerland) unless otherwise stated
Disclaimer
Disclaimer/Publisher’s Note: The statements, opinions and data
contained in all publications are solely those of the individual
author(s) and contributor(s) and not of MDPI and/or the editor(s).
MDPI and/or the editor(s) disclaim responsibility for any injury to
people or property resulting from any ideas, methods, instructions
or products referred to in the content.
Terms and Conditions Privacy Policy
Share Link
Copy
clear
Share

## Page 44

clear
Back to TopTop

