# doi_10_1073-pnas_1600558113

**Source PDF**: doi_10_1073-pnas_1600558113.pdf

---

## Page 1

[image] [image]
•  ARTICLES
Current IssueLatest ArticlesSpecial FeaturesList of IssuesPNAS
Nexus
•  Front Matter
•  AUTHORS
Information for AuthorsEditorial and Journal
PoliciesSubmission ProceduresPublication Charges
•  Topics
○  Physical Sciences
○  Social Sciences
○  Biological Sciences
Featured Topics
○  Physics
○  Chemistry
○  Sustainability Science
○  Sustainable Development Goals
Articles By Topic
○  Applied Mathematics
○  Applied Physical Sciences
○  Astronomy
○  Biophysics and Computational Biology
○  Computer Sciences
○  Earth, Atmospheric, and Planetary Sciences
○  Engineering
○  Environmental Sciences
○  Mathematics
○  Statistics

## Page 2

Featured Topics
○  Anthropology
○  Sustainability Science
○  Sustainable Development Goals
Articles By Topic
○  Demography
○  Economic Sciences
○  Environmental Sciences
○  Political Sciences
○  Psychological and Cognitive Sciences
○  Social Sciences
Featured Topic
○  Anthropology
○  Sustainability Science
○  Sustainable Development Goals
Articles By Topic
○  Agricultural Sciences
○  Applied Biological Sciences
○  Biochemistry
○  Biophysics and Computational Biology
○  Cell Biology
○  Developmental Biology
○  Ecology
○  Environmental Sciences
○  Evolution
○  Genetics
○  Immunology and Inflammation
○  Medical Sciences
○  Microbiology
○  Neuroscience
○  Pharmacology

## Page 3

○  Physiology
○  Plant Biology
○  Population Biology
○  Psychological and Cognitive Sciences
○  Systems Biology
•  
0 Cart
Sign in Register
Individual Login Institutional Login
Submit
Quick Search anywhere
Enter journal, DOI, article type, keywords, authors, etc.
Quick search in Citations
Journal*Journal*PNAS NexusProceedings of the National Academy
of Sciences
YearVolume*Issue*Page/eLocation ID*
*Required field
Quick Search in Journals
Enter journal, DOI, article type, keywords, authors, etc.
Quick Search in Journals
Enter journal, DOI, article type, keywords, authors, etc.
Searching:
Anywhere
AnywhereCitationPNAS NexusProceedings of the National Academy
of Sciences
Advanced Search Search
SUGGESTED SEARCHES:
Covid-19 Artificial Intelligence Mpox Climate Change Gun Violence
Quick Search anywhere
Enter journal, DOI, article type, keywords, authors, etc.
[image]
•  
•  [image]

## Page 4

•  [image]
•  
•  
•  
•  
Contact Site Map Terms & Privacy Policy Accessibility
Research Article
Plant Biology
Free access
Share on
•  
•  
•  
•  
•  
DELAY OF GERMINATION1
(DOG1) regulates both seed
dormancy and flowering time
through microRNA pathways
Heqiang Huo, Shouhui Wei, and Kent J. Bradford
[email protected]Authors Info & Affiliations
Edited by Maarten Koornneef, Max Planck Institute for Plant
Breeding Research, Cologne, Germany, and approved February 24,
2016 (received for review January 13, 2016)
March 28, 2016
113 (15) E2199-E2206
https://doi.org/10.1073/pnas.1600558113
Track CitationsAdd to Reading List
PDF

## Page 5

Contents
Vol. 113 | No. 15
•  Significance
•  Abstract
•  Results
•  Discussion
•  Materials and Methods
•  Acknowledgments
•  Supporting Information
•  References
•  Information & Authors
•  Metrics & Citations
•  View Options
•  References
•  Figures
•  Tables
•  Media
•  Share
Significance
Annual plants adapt to their environments by matching their life
cycles, particularly seed germination and flowering, to the
appropriate seasons. Although genetic evidence has suggested
connections among genes regulating seed dormancy and flowering,
specific mechanisms for such coordination are unknown. We report
that a gene [DELAY OF GERMINATION1 (DOG1)] involved in
determining the depth of seed dormancy, and therefore the seasonal
timing of germination, also influences the timing of flowering in
Arabidopsis and lettuce. We further show that this gene acts through
influencing the production of microRNAs that govern the
progression of developmental phase transitions through the plant
life cycle, providing a molecular genetic mechanism for the
coordinate adaptation of seed dormancy and flowering phenotypes

## Page 6

to environmental conditions.
Abstract
Seed germination and flowering, two critical developmental
transitions in plant life cycles, are coordinately regulated by genetic
and environmental factors to match plant establishment and
reproduction to seasonal cues. The DELAY OF GERMINATION1
(DOG1) gene is involved in regulating seed dormancy in response to
temperature and has also been associated genetically with
pleiotropic flowering phenotypes across diverse Arabidopsis thaliana
accessions and locations. Here we show that DOG1 can regulate
seed dormancy and flowering times in lettuce (Lactuca sativa, Ls)
and Arabidopsis through an influence on levels of microRNAs
(miRNAs) miR156 and miR172. In lettuce, suppression of LsDOG1
expression enabled seed germination at high temperature and
promoted early flowering in association with reduced miR156 and
increased miR172 levels. In Arabidopsis, higher miR156 levels
resulting from overexpression of the MIR156 gene enhanced seed
dormancy and delayed flowering. These phenotypic effects, as well
as conversion of MIR156 transcripts to miR156, were compromised
in DOG1 loss-of-function mutant plants, especially in seeds.
Overexpression of MIR172 reduced seed dormancy and promoted
early flowering in Arabidopsis, and the effect on flowering required
functional DOG1. Transcript levels of several genes associated with
miRNA processing were consistently lower in dry seeds of
Arabidopsis and lettuce when DOG1 was mutated or its expression
was reduced; in contrast, transcript levels of these genes were
elevated in a DOG1 gain-of-function mutant. Our results reveal a
previously unknown linkage between two critical developmental
phase transitions in the plant life cycle through a DOG1–miR156–
miR172 interaction.
Sign up for PNAS alerts.
Get alerts for new articles, or get an alert when an article is cited.
Manage alerts

## Page 7

The life cycles of flowering plants are characterized by distinct
phase transitions such as from seed to seedling (germination) or
from vegetative to reproductive development (flowering) (1). The
timing of germination and flowering both require precise
environmental sensing and integrated responses to multiple inputs
so that developmental transitions can be accurately matched to
seasonal conditions (1–3). Seeds use temperature as a signal of the
seasonal and current environmental conditions to determine
opportune times to germinate with respect to the potential for
seedling survival (2, 4, 5). Similarly, in many plants the transition
from vegetative to floral development occurs in response to
environmental cues, particularly temperature and day length (6, 7).
Ecological and evolutionary studies have found that seed
germination and flowering traits within species are coadapted
across habitat ranges (8–11). Seed dormancy and germination are
regulated primarily by the antagonistic actions of the plant
hormones gibberellin (GA; promotive) and abscisic acid (ABA;
inhibitory), whose synthesis and action vary in response to
environmental signals (12). Recent studies indicate that canonical
genes regulating flowering, such as FLOWERING LOCUS T (FT) and
FLOWERING LOCUS C (FLC), are also involved in the transition
from seed dormancy to germination (13–16), suggesting that seed
dormancy and flowering may be coordinately regulated through
overlapping molecular pathways.
In Arabidopsis, expression of the DELAY OF GERMINATION1 (DOG1)
gene responds to seed maturation temperature and determines the
depth of seed dormancy (2, 3, 5, 17–20). Deeper dormancy of
freshly harvested seeds is associated with high DOG1 transcript
levels, which decrease in after-ripened (nondormant) and
germinating seeds (18, 19). Functional analyses have shown that
DOG1 and ABA are essential for establishing primary seed
dormancy (18, 19, 21), although DOG1 can act independently of
ABA to delay germination of less dormant seeds (20).
Overexpression of DOG1 also increases the sensitivity of seed
germination to inhibition by warm temperatures (5, 20). In
addition, genome-wide association and genetic linkage mapping
studies revealed that the DOG1 locus was associated with flowering
phenotypes across diverse accessions and locations (22, 23).
However, the mechanism of DOG1 action on seed dormancy or
flowering remains unknown.

## Page 8

Lettuce (Lactuca sativa L.) is a major leafy vegetable derived from a
progenitor species (Lactuca serriola L.) having a winter annual
lifecycle, normally flowering in spring after overwintering as a
compact rosette. Lettuce seeds display little primary dormancy, but
germination is strongly inhibited by warm temperatures during
imbibition, a type of relative dormancy termed thermoinhibition
(24, 25), resulting in reduced crop establishment during warm
seasons (26). Previously, we genetically mapped and functionally
characterized the role of L. sativa 9-cis-EPOXYCAROTENOID
DIOXYGENASE4 (LsNCED4), a gene encoding a key regulated
enzyme in ABA biosynthesis, in lettuce seed thermoinhibition (24,
25, 27). Expression of this gene is induced in lettuce seeds by
imbibition at high temperatures, maintaining high ABA content and
preventing germination. A similar role was demonstrated for the
homologous gene Arabidopsis thaliana NCED9 (AtNCED9) (28). The
mechanism by which these genes are regulated by high temperature
remains unknown (29).
Here, we report that suppression of DOG1 expression by RNAi in
lettuce or mutation of DOG1 in Arabidopsis markedly increased the
maximum temperature for seed germination. In addition, DOG1RNAi lettuce plants flowered much earlier than control plants.
Further analyses in lettuce and Arabidopsis showed that these
phenotypes are associated with reduced conversion of primary
MIR156 transcript into the microRNA (miRNA) miR156. Effects of
overexpression of MIR156 on flowering and seed dormancy were
greatly compromised in loss-of-function dog1-3 mutants of
Arabidopsis. The expression of several miRNA processing genes was
altered in seeds of dog1-3 mutants or DOG1-RNAi lines, possibly
associated with the reduced conversion of MIR156 to miR156. In
contrast, a gain-of-function dog1-5 mutation that enhanced seed
dormancy resulted in increased expression of miRNA processing
genes. As have been described previously for flowering (30, 31) and
seed and seedling development (32, 33), follow-on effects of
modification of miR156 levels on miR172 abundance, or direct
overexpression of MIR172, also affected seed dormancy. Our work
provides a potential mechanism for DOG1 action and demonstrates
a previously unknown linkage between seed dormancy and
flowering phenotypes through a DOG1–miR156–miR172
interaction.

## Page 9

Results
DOG1 Influences Seed Thermoinhibition in Lettuce.
To explore whether lettuce homologs of AtDOG1 are involved in
regulation of seed thermoinhibition, cDNAs with homology to
DOG1 were isolated from four genotypes of lettuce having differing
germination thermotolerance. Seeds of cv. Salinas (L. sativa; termed
Sal) and PI261653 (Lactuca saligna; termed Saligna) were unable to
germinate well at 32 °C, whereas seeds of two thermotolerant
genotypes, accession US96UC23 (L. serriola) and PI251246 (L.
sativa; termed PI), germinated fully at this temperature (SI
Appendix, Fig. S1A). Sequences of LsDOG1 cDNAs from the four
genotypes were highly homologous and shared ∼50% overall
protein amino acid sequence similarity with Arabidopsis DOG1
(54.7% similarity within DOG1 domain regions; SI Appendix, Fig. S2
and Table S1). LsDOG1 is highly expressed in mature lettuce seeds,
whereas its transcript level is much lower in leaves and roots (SI
Appendix, Fig. S1C). Its expression in seeds decreased as seed
maturation temperature increased (SI Appendix, Fig. S1D), as is also
the case for AtDOG1 expression in Arabidopsis seeds (17, 18).
Suppression of LsDOG1 expression in Sal lettuce through RNAi
caused 56% and 73% reductions in transcript abundance in two
independent lines (Fig. 1A). Seeds of these DOG1-RNAi lines were
able to germinate fully at 32 °C, whereas seeds of the segregated
nontransgenic control line (i.e., Sal) were completely inhibited (Fig.
1 B and C). In contrast, ectopic expression of LsDOG1 under the
native Sal DOG1 promoter (PDOG1:LsDOG1) in the thermotolerant PI
lettuce line resulted in increased DOG1 transcripts in mature dry
seeds and enhanced seed thermoinhibition at 30 °C (Fig. 1 D–F). A
functional homolog of DOG1 is therefore present in lettuce and is
involved in regulating germination thermoinhibition.
Fig. 1.
[image]Regulation by LsDOG1 of lettuce seed thermoinhibition. (A)
Relative LsDOG1 mRNA levels in dry seeds of Sal lettuce and its
LsDOG1-RNAi lines, 21–5G and 21–1I. (B and C) Seed germination
of Sal, 21–5G, and 21–1I lines at 20 and 32 °C. (D) Relative LsDOG1
mRNA levels in dry seeds of PI lettuce and its LsDOG1 overexpression lines, 33–10 and 33–04. (E and F) Seed germination of

## Page 10

PI, 33–10, and 33–04 lines at 20 and 30 °C. In D and F, seeds at 20
°C were imbibed for 16 h, whereas seeds at 32 and 30 °C were
imbibed for 30 h. Error bars represent SE (n = 3).
DOG1 Influences Seed Thermoinhibition in
Arabidopsis.
Functional DOG1 action in Arabidopsis promotes seed dormancy and
extends after-ripening periods required for dormancy alleviation
(18, 19), whereas fresh seeds of loss-of-function dog1 mutants can
fully germinate without after-ripening or cold stratification (17,
19). To test whether DOG1 might be involved in regulating
germination thermoinhibition in Arabidopsis, we imbibed seeds of
the Columbia (Col) dog1-3 mutant (Salk_000867) at 32 °C. Mutant
seeds germinated more than 80% whereas WT seeds germinated
less than 20% at that temperature (SI Appendix, Fig. S1B).
Molecular complementation showed that coding regions of LsDOG1
genes from thermosensitive and thermotolerant lettuce genotypes
were able to rescue the thermoinhibition phenotype when expressed
in Arabidopsis dog1-3 plants (SI Appendix, Fig. S1B), indicating that
these homologs are all functional and that their role in regulation of
seed thermoinhibition is conserved. The presence of functional
DOG1 alleles in US96UC23 and PI251246 is consistent with lack of
association of this locus with quantitative trait loci for hightemperature germination in these genotypes (27, 34).
Previous work in lettuce and Arabidopsis showed that NCED genes
encoding a key enzyme in the ABA biosynthetic pathway are
involved in thermoinhibition of germination (24, 28). We tested the
interactive effects of DOG1 and ABA on seed thermoinhibition by
overexpressing LsDOG1 in an atnced6-1 atnced9-1 double mutant
that displays strong germination thermotolerance as a result of
reduced ABA biosynthesis (24, 28). Overexpression of LsDOG1
restored a thermosensitive phenotype to atnced6-1atnced9-1 seeds at
32 °C (SI Appendix, Fig. S1E), indicating that DOG1 and ABA can
induce thermoinhibition via parallel pathways. We also generated a
dog1-3 × nced9-1 double mutant to further examine the interactive
effect of both genes on thermoinhibition. At a higher temperature
(35 °C), seeds of the single mutants of dog1-3 and nced9-1 and of the
double mutant nced6-1 nced9-1 were fully inhibited from
germinating, whereas dog1-3 nced9-1 seeds germinated 95% (SI

## Page 11

Appendix, Fig. S1F). This additional effect of dog1-3 in the presence
of ABA biosynthesis mutants is in contrast to reports that ABA
biosynthesis and functional DOG1 are required for induction of
primary dormancy (18, 19). However, consistent with previous
reports (17–20), overexpression of LsDOG1 under the CaMV35S
promoter in Arabidopsis Col plants resulted in deep seed dormancy
that could not be alleviated by the ABA biosynthesis inhibitor
fluridone, but could be alleviated by GA (SI Appendix, Fig. S3).
Thus, DOG1 and ABA can regulate seed primary dormancy and
thermoinhibition, and may act in concert or independently
depending on the conditions.
DOG1 Influences Flowering Time in Lettuce Through
Effects on miR156.
Suppression of DOG1 expression in Sal lettuce caused early
flowering (Fig. 2 A–C), as occurred also in wheat (35). Plants from
seeds of seven independent transgenic DOG1-RNAi lines that
germinated at 32 °C and from segregated control (nontransgenic)
seeds were grown in the greenhouse under relatively noninductive
day-length conditions for flowering in fall 2013. Salinas DOG1RNAi lines flowered in ∼120–150 d, whereas control lines did not
flower until >200 d (Fig. 2B). Similar results were observed in two
DOG1-RNAi lines that were grown in spring 2014, although these
plants all flowered earlier because of lengthening days (Fig. 2C).
Fig. 2.
[image]Regulation of flowering time in lettuce by DOG1. (A) Two
independent early-flowering lettuce lines (DOG1-RNAi) and the
control Sal line (CTL) are shown at 120 d after seeding. (B) Times
to flowering of F2 plants of seven independent transgenic lines and
a segregated control line (CTL). Plants were sown in September
2013 and were genotyped for the presence of the transgene (n ≥ 20
plants per line). (C) Times to flowering of two homozygous DOG1RNAi lines and a control line that were sown in late February 2014
(n ≥ 20 plants per line). (D) Relative levels of miR156 and miR172
and mRNA levels of SPL3, SPL4, and SPL9 in apical meristems of 6wk-old control (CTL) and DOG1-RNAi (21-5G, 21–1I) lettuce plants;
levels were first normalized by LsUBQ10 and are shown relative to
those in control for each gene or miRNA. Error bars represent SE (n
= 3).

## Page 12

To assess which flowering regulatory pathways were affected by
suppression of LsDOG1 expression, transcript levels of key genes in
the different pathways were investigated (SI Appendix, Fig. S4).
Consistent with early flowering, transcripts of the floral promoter
FT were almost 25-fold higher in young DOG1-RNAi lettuce leaves
than in similar leaves of the control line (SI Appendix, Fig. S4).
Expression patterns of other flowering-related genes were not
consistent with effects on the photoperiod-dependent, GA,
vernalization, or autonomous flowering pathways (SI Appendix,
Supplementary Text and Figs. S4 and S5). However, transcripts of
SQUAMOSA PROMOTER BINDING PROTEIN-LIKE (SPL) 3, 4, and 9
were increased in leaves of the DOG1-RNAi line compared with its
control line (SI Appendix, Fig. S4). Furthermore, the relative mRNA
levels of SPL3 and 4 were significantly elevated in apical meristems
of DOG1-RNAi lines, although SPL9 expression was not affected
(Fig. 2D). SPL gene expression is inhibited by DELLA proteins (SI
Appendix, Fig. S5), so a decrease in them could result in increased
SPL expression (36). However, transcripts of genes encoding two
major DELLA proteins (RGL1 and RGA) were unchanged or elevated
in DOG1-RNAi leaves (SI Appendix, Fig. S4), making it unlikely that
this is the cause of increased SPL expression. SPL transcripts are
targeted for degradation by miR156, a regulator of developmental
phase transitions in plants, and SPL proteins can induce expression
of FT and other genes promoting flowering (30). SPLs also promote
production of miR172, a positive regulator of maturation (juvenile
to adult) and floral phase transitions (30). Thus, if this mechanism
is operative, we would expect early flowering and high SPL
transcript levels to be associated with lower miR156 and higher
miR172 levels. Consistent with this, miR156 abundance in apical
meristems of DOG1-RNAi lines was only one third of that in the
control line, whereas miR172 levels were approximately doubled
(Fig. 2D). Thus, the early flowering in DOG1-RNAi lettuce plants
may be a result of a reduction in miR156 levels, which would result
in increased levels of the floral-promoting SPLs and of miR172 (SI
Appendix, Fig. S5).
DOG1 Can Influence Flowering in Arabidopsis.
In contrast to early flowering when LsDOG1 expression was
suppressed in lettuce, no effects on flowering times were observed

## Page 13

in Arabidopsis dog1-3 mutant plants under long-day or short-day
conditions (Table 1 and SI Appendix, Table S2). However, when
miR156 levels were elevated by overexpression of the gene
encoding it, MIR156 (Col-35S:LsMIR156), flowering times were
significantly delayed (Fig. 3A), with a majority of plants beginning
to flower more than 50 d after germination (DAG) and having more
than 36 rosette leaves; in contrast, flowering of homozygous dog1-3
plants also expressing 35S:LsMIR156 was only slightly delayed
compared with Col and dog1-3 plants (Table 1 and SI Appendix,
Table S2). Further evaluation of 48 dog1-3–35S:LsMIR156 and 72
Col-35S:LsMIR156 independent transgenic lines showed that 87% of
the former had flowered within 50 DAG and 95% had less than 30
leaves at flowering, whereas 72% of the latter required more than
50 d to flower and 83% of flowering lines had more than 30 leaves
(Table 1 and SI Appendix, Table S2). Although it is possible that
sequence homology between the CaMV35S promoter and the TDNA insertion in dog1-3 could silence the transgene (37), this result
was also reproduced in Landsberg erecta (Ler) and its γ-irradiation–
induced dog1-1 mutant (19), although the delay in flowering time
caused by 35S:LsMIR156 in Ler was not as strong as in Col (Table 1
and SI Appendix, Table S2). As in Arabidopsis, overexpression of
LsMIR156 also greatly delayed flowering in Sal lettuce (SI Appendix,
Fig. S6). To test whether this phenomenon is specific to lettuce
MIR156, we crossed dog1-3 with a Col-35S:AtMIR156 line that
exhibits delayed flowering (38). Effects on flowering were
essentially identical to those for LsMIR156 (Fig. 3A, Table 1, and SI
Appendix, Table S2). The lack of response of dog1-3 plants to
overexpression of LsMIR156 or AtMIR156 was evidently caused by
reduced conversion of pri-MIR156 transcripts to miR156, as
LsMIR156 (or AtMIR156) transcripts were present at equal or
greater levels in dog1-3 meristems (Fig. 3B and SI Appendix, Fig.
S7F) or leaves (SI Appendix, Fig. S7A) as in Col tissues also
expressing LsMIR156 or AtMIR156. The levels of miR156 in dog1-3–
35S:LsMIR156 meristems were only 28% or 13% of those in 25-dold apical meristems or leaves, respectively, of the
Col-35S:AtMIR156 plants (Fig. 3B and SI Appendix, Fig. S7B). This
was associated with an increase in miR172 and SPL3, 4, 5, and 9
transcripts in dog1-3–35S:LsMIR156 meristems and leaves (Fig. 3B
and SI Appendix, Fig. S7 C–E). Similar differences in miR156 and
miR172 levels were also observed in Col and dog1-3 meristems in

## Page 14

response to overexpression of AtMIR156 (SI Appendix, Fig. S7F).
Table 1.
DOG1 effect on flowering times (days to flowering) of Arabidopsis
genotypes
Experiment
Hm/
Ht
PN
Interval
and
incidence
of
flowering
plants
Experiment 1 (LD, 22
°C, 135
µmol⋅m−2⋅s−1)
Days to flowering

21–25
26–
30
31–
35
36–
40
41–
50
51–
60
61
70

Flowering
plants, %
 Col
Hm
72
6.9
91.7 1.4
—
—
—
—
 Col-35S:LsMIR156-G Hm
65
—
—
—
—
1.5
32.3 27
 Col-35S:LsMIR156-C Hm
53
—
—
—
—
28.3
64.1 1.
 Col-35S:AtMIR156
Hm
70
—
—
—
—
—
7.1
15
 dog1-3
Hm
61
3.3
55.7 41.0  
—
—
—
 dog1-3–
35S:LsMIR156-I
Hm
69
—
27.5 58.0 14.5 —
—
—
 dog1-3–
35S:LsMIR156-G
Hm
68
—
11.8 83.8 4.4
—
—
—
 dog1-3 ×
35S:AtMIR156-A
Hm
63
—
9.5
41.3 49.2 —
—
—
 dog1-3 ×
35S:AtMIR156-G
Hm
65
—
6.2
76.9 16.9 —
—
—
 nced9-1
Hm
48
—
77.1 22.9 —
—
—
—
 
nced9-1×AtMIR156#7
Hm
40
—
—
—
—
2.5
5
17

## Page 15

Experiment 2 (LD, 21
°C, 100
µmol⋅m−2⋅s−1)
Days to flowering

21–25
26–
30
31–
35
36–
40
41–
50
51–
60
61
70

Flowering
plants, %
 Col
Hm
48
—
66.7 33.3 —
—
—
—
 Col-35S:LsMIR156
Ht
72
—
—
—
1.4
26.4
8.3
5.
 dog1-3
Hm
36
—
41.7 58.3 —
—
—
—
 dog1-3–
35S:LsMIR156
Ht
48
—
—
8.4
33.3 45.8
12.5 —
 dog1-5
Hm
20
—
55
45
—
—
—
—
 dog1-5–
35S:LsMIR156
Ht
28
—
—
—
—
7.1
7.1
21
 nced9-1
Hm
47
—
56.3 43.7 —
—
—
—
 nced9-1–
35S:LsMIR156
Ht
37
—
—
—
5.4
16.2
8.1
16
Experiment 3 (LD, 22
°C, 135
µmol⋅m−2⋅s−1)
Days to flowering

21–25
26–
30
31–
35
36–
40
41–
50
51–
60
61
70

Flowering
plants, %
 Ler
Hm
48
20.8
79.2 —
—
—
—
—
 Ler-35S:LsMIR156
Ht
63
—
4.8
14.3 6.3
55.5
14.3 4.
 dog1-1
Hm
57
17.5
82.5 —
—
—
—
—
 dog1-1–
35S:LsMIR156
Ht
52
—
40.4 50
9.6
—
—
—
Experiment 4 (SD, 22
°C, 135
µmol⋅m−2⋅s−1)

## Page 16

Days to flowering

61–70
71–
80
81–
90
91–
100
>100

Flowering
plants, %
 Col
Hm
37
—
—
10.8 32.4 56.8
 Col-35S:LsMIR172
Ht
77
10.4
10.4 18.2 44.2 16.8
 dog1-3
Hm
64
—
—
9.5
29.7 60.8
 dog1-3–
35S:LsMIR172
Ht
100 —
1.0
8.0
42.0 49.0
Hm, homozygous; Ht, heterozygous; LD, long day; PN, number of
plants when homozygous or number of independent transgenic
individuals when heterozygous; SD, short day. Dashes indicate
absence of plants in that category.
Fig. 3.
[image]Regulation of flowering time in Arabidopsis by DOG1. (A)
Seven-week-old Col, dog1-3, dog1-3–35S:LsMIR156 (dog1-Ls156),
dog1-3 × Col-35S:AtMIR156 (dog1×At156), Col-35S:LsMIR156
(Col-Ls156), and Col-35S:AtMIR156 (Col-At156) plants grown in
long days. (B) Relative levels of miR156 and miR172 and mRNA
levels of LsMIR156, SPL3, SPL4, SPL5, and SPL9 in apical meristems
of Col and dog1-3 plants expressing LsMIR156 at 25 DAG; levels
were first normalized by AtACT2, and values relative to those in the
Col-LsMIR156 meristems for each gene or miRNA are shown. Error
bars represent SE (n = 3). (C) Col-35S:LsMIR172 and dog1-3–
35S:LsMIR172 plants grown for 65 d in short days.
It was recently reported that dog1-5 (SALK_022748) is a gain-offunction mutant in which the level of DOG1 protein is greatly
increased and seed dormancy is enhanced compared with Col WT
(39), as we have confirmed here (SI Appendix, Fig. S8). In this case,
the mutation in DOG1 did not negate the effect of MIR156
overexpression on delaying flowering, but rather somewhat
enhanced it; 93% of independent dog1-5-LsMIR156 transgenic lines
required more than 50 DAG to flower compared with 72% of ColLsMIR156 plants, with similar data for numbers of leaves at
flowering (Table 1 and SI Appendix, Table S2). Interestingly, as for
dog1-3, we did not observe a significant difference in flowering time
for dog1-5 plants in the absence of MIR156 overexpression (Table 1
and SI Appendix, Table S1). Thus, in contrast to lettuce, DOG1

## Page 17

regulation of flowering time in Arabidopsis is evident only when
MIR156 expression is enhanced.
ABA has also been reported to be involved in floral transitions (40),
raising the question of whether the early flowering observed in
lettuce DOG1-RNAi plants and dog1-3 plants overexpressing MIR156
could be caused by the alteration of ABA contents, as mutants in
ABA biosynthetic genes have similar seed germination phenotypes
as dog1-3 mutant seeds (SI Appendix, Fig. S1 B and E). However, the
delayed flowering time phenotypes conferred by 35S:AtMIR156 and
35S:LsMIR156 in Col plants also occurred in an nced9-1 mutant
background (Table 1 and SI Appendix, Table S2). Thus, although
phenotypic effects of MIR156 overexpression on flowering depend
upon functional DOG1, expression of NCED9, which is primarily
seed-expressed and required for thermoinhibition (28), is not
necessary.
The results reported here show that loss of DOG1 function
negatively affects processing of MIR156 to miR156 in lettuce and
Arabidopsis and elevates miR172 abundance. As miR172 is known
to promote early flowering in Arabidopsis (30), we tested the effect
of overexpression of LsMIR172 on flowering in Col WT and dog1-3
mutant plants. To maximize phenotypic differences, we grew 100
(for Col-35S:LsMIR172) and 77 (for dog1-3–35S:LsMIR172)
independent transgenic lines under short-day conditions. Under
short days, WT Col plants developed more than 40 leaves before
flowering and 90% of plants did not flower before 90 d, whereas
85% of Col-35S:MIR172 plants had less than 40 leaves at flowering
and 39% started flowering in less than 90 DAG (Fig. 3C, Table 1,
and SI Appendix, Table S2). This advancement of flowering as a
result of overexpression of MIR172 was much reduced in dog1-3
plants: more than 60% of dog1-3–35S:LsMIR172 plants had more
than 40 leaves at flowering, as did the dog1-3 plants, and only 9%
of dog1-3:35S:LsMIR172 plants flowered in less than 90 DAG (Fig.
3C, Table 1, and SI Appendix, Table S2). Thus, functional DOG1
plays a role in the processing or action of MIR172 as well as of
MIR156 (SI Appendix, Fig. S5).
miR156 and miR172 Affect Seed Germination.
Suppression of DOG1 expression in lettuce, which improved
germination at warm temperatures (Fig. 1 B and C), also resulted in

## Page 18

large reductions in miR156 content in dry seeds, with levels only 5–
7% of that in control seeds (Fig. 4A). In mature dog1-3 mutant
seeds, the miR156 level was slightly lower than that in mature WT
Col seeds, but this difference was more significant when LsMIR156
was overexpressed (Fig.4B). We therefore tested whether expression
of MIR156 and MIR172 would affect primary dormancy and
thermoinhibition of Arabidopsis seed germination in WT and dog1-3
mutant plants. Expression of 35S:LsMIR156 or 35S:AtMIR156 in Col
plants reduced germination at 25 °C, likely reflecting increased
primary dormancy, and strongly enhanced seed thermoinhibition at
warmer temperatures (Fig. 4C). In contrast, overexpressing
LsMIR156 in dog1-3 plants did not affect germination at 25, 30, or
32 °C (Fig. 4C). Although seeds of the nced9-1 mutant, like dog1-3
seeds, were thermotolerant, expression of 35S:AtMIR156 in nced9-1
plants had similar effects in suppressing germination as in Col
plants (Fig. 4C). This demonstrates that elevation of miR156 can
inhibit germination independently of seed ABA biosynthesis and
that the effect of MIR156 expression on seed germination inhibition
is dependent on functional DOG1. In contrast, overexpression of
LsMIR172 promoted germination of fresh (primary dormant) Col
seeds, even at elevated temperatures (Fig. 4D). As dog1-3 seeds did
not exhibit primary dormancy and germinated fully at all
temperatures tested, an additional effect of LsMIR172
overexpression on germination of dog1-3 seeds could not be
detected (Fig. 4D). Thus, miR156 and miR172 can influence seed
thermoinhibition as well as flowering and other developmental
phase transitions (SI Appendix, Fig. S5).
Fig. 4.
[image]Lettuce seed content of miR156, effects of modification of
miR156 or miR172 expression on germination, and effects of dog1
mutations on expression of DICER-related genes. (A) Levels of
miR156 in dry seeds of control and three homozygous DOG1-RNAi
lettuce lines, 21–6A, 21–5G, and 21–1I. (B) miR156 levels in dry
seeds of Col, dog1-3 mutant, Col expressing LsMIR156, and dog1-3
mutants expressing LsMIR156. (C) Effect of expression of
35S:AtMIR156 and 35S:LsMIR156 on germination of seeds of Col,
nced9-1, and dog1-3 genotypes of Arabidopsis. All seeds were tested
at 3 wk after harvesting. (D) Germination at 25, 30, or 32 °C of Col
WT or dog1-3 mutant seeds and of seeds of these genotypes that had
been transformed to express 35S:LsMIR172. Seeds were tested 5 d

## Page 19

after harvesting when primary dormancy was still present in ColWT seeds to demonstrate alleviation of dormancy by LsMIR172
overexpression. (E–G) Transcripts of genes associated with miRNA
processing or transcription (DCL1, HYL1, TGH, SE, and CDC5) in
dry seeds of Col and its T-DNA dog1-3 (loss-of-function) and dog1-5
(gain-of-function) mutants (E), of Ler-CviDOG1 introgression line
and its irradiation-induced mutant dog1-1 (F), and of Salinas lettuce
(CTL) and its DOG1-RNAi silencing line (21–1I) (G). mRNA levels
were first normalized by AtACT2; values relative to those in Col (E),
in Ler-CviDOG1 (F), and in Sal (G) are shown. Error bars represent
SE (n = 3).
Altered DOG1 Expression or Function Affects
Expression of Genes Encoding miRNA Processing
Proteins.
As the conversion of primary MIR156 to miR156 was greatly
compromised in seeds of lettuce DOG1-RNAi suppression lines and
of Arabidopsis dog1-3 mutants (Fig. 4 A and B), we tested whether
transcripts of genes encoding proteins associated with miRNA
transcription or processing were altered. After preliminary
screening, we focused on five genes: DICER-LIKE 1 (DCL1),
HYPONASTIC LEAVES1 (HYL1), the G-patch domain protein
TOUGH (TGH), the zinc finger protein SERRATE (SE), and CELL
DIVISION CYCLE 5 (CDC5) that binds RNA polymerase II and MIR
promoters to positively regulate transcription of MIR-encoding
genes (41, 42). Although the specific effects varied somewhat
among the Arabidopsis and lettuce genotypes and mutants tested,
transcript levels of all of these genes differed from their respective
WTs in at least some cases (Fig. 4 E–G). For example, the transcript
levels of DCL1, HYL1, and CDC5 in dry dog1-3 mutant seeds were
56%, 55%, and 76%, respectively, of those in dry Col WT seeds,
whereas expression of these genes (and of TGH and SE) was
elevated compared with Col-WT in seeds of the gain-of-function
dog1-5 mutant (Fig. 4E). Consistent reduction in expression of these
genes was also observed in dry seeds of the dog1-1 mutant
compared with those in Ler-CviDOG1 (Fig. 4F), which contains a
strong CviDOG1 allele (19), and transcripts of DCL1, SE, TGH, and
CDC5 were also lower in dry DOG1-RNAi lettuce seeds (Fig. 4G).

## Page 20

These results indicate that DOG1 could affect miR156 (and possibly
miR172) processing in seeds through altering expression of genes
encoding miRNA-processing proteins (SI Appendix, Fig. S5).
Discussion
Seed germination and flowering are two critical developmentalphase transitions in the plant life cycle. Plants sense environmental
cues such as temperature to coordinate endogenous genetic and
molecular mechanisms that determine the timing of both phase
transitions (1, 6, 7, 12, 43, 44). Although its mechanism of action
remains unknown, DOG1 has been demonstrated to be a positive
regulator for the establishment and maintenance of primary seed
dormancy in several species (19, 20, 35). DOG1 is involved in
transducing maternal environmental conditions during seed
development into effects on seed dormancy and in the loss of
dormancy through dry after-ripening or moist chilling (2, 5, 17, 45,
46). ABA is also a strong positive regulator of seed dormancy (12)
and promoted DOG1 expression in imbibed Lepidium sativum seeds
(47). Arabidopsis seeds may require ABA and functional DOG1 to
establish primary dormancy, as reduced dormancy in Arabidopsis
seeds was still observed in ABA-deficient aba1 mutants, even when
the strong Cvi-DOG1 allele was present (18, 19), whereas seeds of a
dog1-2 cyp707a2-1 double mutant that have higher levels of ABA
nonetheless exhibited reduced seed dormancy as a result of the loss
of DOG1 function (18). Here, we demonstrated that DOG1 is
involved in regulation of thermoinhibition of seed germination in
lettuce and Arabidopsis, but, unlike the dependence of primary
dormancy on ABA and DOG1, DOG1 can act in the absence of
enhanced ABA synthesis to impose thermoinhibition, consistent
with the relative ineffectiveness of an ABA biosynthesis inhibitor to
alleviate effects of overexpression of DOG1 on thermoinhibition (SI
Appendix, Fig. S3) (20). Thus, ABA and DOG1 may regulate seed
thermoinhibition through parallel but mutually reinforcing
pathways that may not be identical to those by which they act on
primary dormancy (29). On the contrary, GA can alleviate the
effects of overexpression of DOG1 on thermoinhibition (SI Appendix,
Fig. S3) (20), as might be expected from the suppression of

## Page 21

expression of GA biosynthetic enzymes by high temperatures in
Arabidopsis and lettuce (25, 28, 34).
Ecological genetics studies have shown that, under natural seasonal
variation, DOG1 can influence seed germination and flowering
times (48). Although, initially, the effects of DOG1 on flowering
were suggested to be an indirect response to the seasonal timing of
germination, further study showed that the alleviation of seed
dormancy resulted in an acceleration of flowering that was
independent of germination timing (49). Genome-wide association
and genetic linkage mapping analyses have implicated the DOG1
locus in the regulation of flowering phenotypes (22, 23), but these
results were attributed to the possibility of being caused by closely
linked genes, as DOG1 was assumed to be active only in seeds.
However, our results demonstrate that DOG1 itself can have effects
on seed dormancy and flowering, suggesting a more direct
mechanism for the evolutionary coadaptation of life-cycle
transitions to match seasonal environmental conditions (16, 44, 49,
50). In addition, previous studies have shown that genes related to
flowering are expressed in association with seed development and
germination. For example, a genome-wide expression correlation
network in Arabidopsis showed that maturity-phase transition
regulators such as SE, EMBRYONIC FLOWER1, and EARLY BOLTING
IN SHORT DAYS are coexpressed with germination-associated genes
and affect seed germination (14). Arabidopsis HISTONE
MONOUBIQUITINATION1 and 2 (HUB1 and HUB2) genes regulate
flowering and seed germination; seeds of hub1 and hub2 mutants
displayed reduced seed dormancy (51), and hub1 seeds also exhibit
strong thermotolerance to high temperature at germination (52).
The increased germination of hub1 and hub2 seeds could be
attributed to the altered expression of several seed dormancyrelated genes, including DOG1 and NCED9 (51). Loss of function of
HUB1 and HUB2 also caused early flowering phenotypes through
repression of FLC by chromatin modification (53). In addition to
repressing flowering, FLC also promotes seed germination at low
temperature (10 °C) via changing expression of ABA-catabolism
(CYP707A2) and GA-biosynthesis (GA20ox1) genes (15). Consistent
with these observations, the flowering-promotive protein FT
maternally controls seed dormancy through alteration in seed coat
tannin content in Arabidopsis (13). Thus, although there have been
previous indications that seed dormancy and flowering might share

## Page 22

common genetic and molecular mechanisms, our understanding of
how such mechanisms might be connected to each other and to
environmental cues has been limited.
Our results demonstrate that DOG1 regulates seed germination and
flowering time at least in part through an influence on generation
and/or action of miR156 and miR172. miRNAs have been
implicated previously as being involved separately in seed
germination and flowering time (30, 32). Mutants that are defective
in DCL1, HYL1, and SE, all of which are required for miRNA
biogenesis (41), display abnormal flowering and alteration in ABA
sensitivity of seed germination (54, 55). Interestingly, the delayedflowering phenotype of dcl1-7 mutant could be rescued by
expression of a DCL4-dependent MIR839 in which the mature
sequence was replaced by the one encoding MIR172, suggesting
that the delayed flowering in dcl1-7 plants is a result of the
reduction in miR172 (56). Earlier transition from the juvenile to the
adult phase was also observed in loss-of-function mutants of SE and
HYL1, in association with substantial reduction in miR156, and
could be rescued by overexpression of MIR156 (57, 58). Another
miR156 target, SPL13, plays an important role in the regulation of
the postgerminative switch from the cotyledon stage to the
vegetative-leaf stage during seedling growth (59). Mutants in dcl1
exhibit early seed maturation programming, probably caused by
increased levels of SPL10 and SPL11 as a consequence of reduced
miR156 (60, 61). Our data indicate that DOG1 may affect the
conversion of primary MIR156 to miR156 or primary MIR172 to
miR172 through an effect on expression of genes involved in
miRNA processing (SI Appendix, Fig. S5). Loss of function or gain of
function of DOG1 decreases or increases, respectively, expression of
DICER-related genes (Fig. 4 E–G), and previous studies found that
miR156 and miR172 levels are altered in mutants of DCL1, HYL1,
SE, TGH, and CDC5 (42, 56, 58, 62, 63). However, multiple
feedback loops are associated with transcription and processing of
miRNAs (41), so further study is needed to clarify the specific
mechanism by which DOG1 affects miRNA levels. In addition, the
strong effects of DOG1 overexpression in enhancing seed dormancy
even in the absence of MIR156 overexpression (SI Appendix, Fig. S3)
(18, 19) indicate that DOG1 may also act on seed germination
through additional mechanisms (SI Appendix, Fig. S5).
miR156 accumulates to high levels during seed development (32,

## Page 23

64) and is abundant in young seedlings and decreases as plants age,
in association with the transition to adult phase and flowering (30,
38, 65). This pattern indicates that high miR156 levels are reset
during embryogenesis and seed development, as occurs for FLC, a
strong floral repressor (7). Winter annual flowering plants require
prolonged chilling to epigenetically silence FLC to flower the next
spring; to ensure the requirement for vernalization in every
generation, it is essential to reactivate FLC gene expression during
seed development (66). Consistent with this, expression and
maintenance of FLC during late embryogenesis are required for late
flowering phenotypes (67, 68). Similarly, DOG1 may play a role in
resetting high miR156 levels during seed development (Fig. 4A),
and defects in DOG1 function could result in altered
postgerminative plant development and floral transition as well as
reduced seed dormancy (SI Appendix, Fig. S5) (32, 41, 64, 65). We
also note that a primary phenotype of DOG1 action is an extension
of dry after-ripening times required for dormancy alleviation (19).
A role for miRNAs produced during seed development in
maintaining seed dormancy provides a mechanism for the afterripening effect, in which seeds lose dormancy during dry storage
when metabolism is prevented (12). miR156 accumulation during
seed development, and its subsequent loss during storage as a result
of oxidation, as shown for stored mRNAs (69), would provide a
timing mechanism for after-ripening via loss of the inhibitory effect
of stored miR156 on germination.
Our results support the conclusion that the transition from
dormancy to germination is a life-cycle phase transition comparable
to vegetative maturation or flowering, and apparently is influenced
by the same miRNA-dependent systems (SI Appendix, Fig. S5) (32).
Effects on these systems are also consistent with the hypothesis that
DOG1 primarily affects the underlying environmental sensitivity of
processes it influences, rather than directly determining final
phenotypes (5). As the miR156/SPL/miR172 pathway is involved in
modulating the inputs from a diverse array of environmental signals
and transducing them into developmental responses (6, 7, 32, 59),
interaction of DOG1 with this system would enable it to play an
integrating role by matching the environmental sensitivities of seed
dormancy and flowering to current or anticipated conditions. A
dual role in sensing environmental signals (e.g., temperature) and
coordinating developmental-phase transitions in the plant life cycle

## Page 24

would explain the repeated identification of DOG1 as a significant
locus in ecological genetics studies of flowering phenotypes (22, 23,
48). It would also make evolutionary sense, as seed dormancy and
germination timing influence the environment in which subsequent
flowering and reproduction occur, and vice versa (16, 44, 50). An
integrated mechanism for coordinating these two major life cycle
transitions having significant impacts on fitness and survival would
be subject to coselection to optimize (or bet-hedge) both (9, 11, 43,
49, 50). Much remains to be done to fully understand these
complex interactions between plant life cycles and the environment,
but our results demonstrate that DOG1 functions as an important
molecular integrator that exerts its effects on developmental phase
changes at least in part through miRNA-regulated pathways.
Materials and Methods
Seeds of lettuce (L. sativa L.) were germinated at room temperature
and then transferred to an automatically controlled greenhouse at
23 ± 2 °C (day) and 20 ± 2 °C (night) in long days (14 h light/10
h dark) for seed production unless stated otherwise. Seeds of A.
thaliana were stratified at 4 °C for 2–4 d before being transferred to
a growth chamber at 21–22 °C under long days (16 h light/8 h
dark, 100 or 135 µmol⋅m−2⋅s−1 light intensity) or short days (8 h
light/16 h dark, 135 µmol⋅m−2⋅s−1 light intensity) for seed
production. Seeds of the γ-irradiation–induced dog1-1 mutant were
provided by Leonie Bentsink, Wageningen University, Wageningen,
The Netherlands (19), seeds of the T-DNA mutant nced9-1 by Eiji
Nambara, University of Toronto, Toronto, and seeds of the T-DNA
double-mutant nced6-1 nced9-1 by Annie Marion-Poll, Institut JeanPierre Bourgin, Versailles, France. Seeds of Arabidopsis WT (Col and
Ler), T-DNA dog1-3 mutant (Salk_000867) (19), T-DNA dog1-5
mutant (SALK_022748) (39), and AtMIR156-overexpression line
(CS67849) (30) were purchased from the Arabidopsis Biological
Resource Center. The dog1-3 mutant line was selfed and genotyped
for three generations before this study. The Arabidopsis T-DNA
mutants were genotyped using the primers listed in SI Appendix,
Table S3. Details of experimental procedures including isolation of
LsDOG1 and vector construction, generation of dog1-3 ×

## Page 25

35S:AtMIR156 and nced9–1 × 35S:LsMIR156 lines, seed
germination assays, measurements of flowering times, identification
of LsMIR156 and LsMIR172, vector construction for overexpression
of LsMIR156 and LsMIR172, plant transformation, and mRNA and
miRNA analyses are described in SI Appendix, Supplementary
Methods. Primers used in this study are listed in SI Appendix, Table
S3.
Acknowledgments
We thank University of California, Davis, student interns Phat Hua,
Alex Wai, and Macrae Dec-Hull and visiting Universidade Federal
de Viçosa (Brazil) student Danubia Nobre for assistance in
maintaining plants. Extramural funding for this work was provided
by United States Department of Agriculture (USDA)-National
Institute of Food and Agriculture Grant 2008-02509, USDACooperative State Research, Education, and Extension Service
Regional Research Project W3168, and Rijk Zwaan B.V., De Lier,
The Netherlands. Access to lettuce genomic sequences was provided
through the University of California, Davis, Genome Center and
supported in part by National Science Foundation Award 0820451
and the Compositae Genome Project. The visit of S.W. to University
of California, Davis, was supported by China Scholarship Council
(201203250049) and Natural Science Foundation of China
(31171867).
Supporting Information
Appendix (PDF)
Supporting Information
•  Download
•  698.77 KB
References

## Page 26

1
P Huijser, M Schmid, The control of developmental phase
transitions in plants. Development 138, 4117–4129 (2011).
Crossref
PubMed
Google Scholar
2
S Footitt, Z Huang, HA Clay, A Mead, WE Finch-Savage,
Temperature, light and nitrate sensing coordinate Arabidopsis seed
dormancy cycling, resulting in winter and summer annual
phenotypes. Plant J 74, 1003–1015 (2013).
Crossref
PubMed
Google Scholar
3
GCK Chiang, et al., DOG1 expression is predicted by the seedmaturation environment and contributes to geographical variation
in germination in Arabidopsis thaliana. Mol Ecol 20, 3336–3349
(2011).
Crossref
PubMed
Google Scholar
4
D Batlla, R Benech-Arnold, A framework for the interpretation of
temperature effects on dormancy and germination in seed
populations showing dormancy. Seed Sci Res 25, 147–158 (2015).
Crossref
Google Scholar
5
M Murphey, et al., DOG1-imposed dormancy mediates germination
responses to temperature cues. Environ Exp Bot 112, 33–43 (2015).
Crossref
Google Scholar
6
F Andrés, G Coupland, The genetic basis of flowering responses to
seasonal cues. Nat Rev Genet 13, 627–639 (2012).
Crossref
PubMed
Google Scholar
7

## Page 27

R Amasino, Seasonal and developmental timing of flowering. Plant J
61, 1001–1013 (2010).
Crossref
PubMed
Google Scholar
8
C Alonso-Blanco, H Blankestijn-de Vries, CJ Hanhart, M Koornneef,
Natural allelic variation at seed size loci in relation to other life
history traits of Arabidopsis thaliana. Proc Natl Acad Sci USA 96,
4710–4717 (1999).
Crossref
PubMed
Google Scholar
9
PE Toorop, et al., Co-adaptation of seed dormancy and flowering
time in the arable weed Capsella bursa-pastoris (shepherd’s purse).
Ann Bot (Lond) 109, 481–489 (2012).
Crossref
PubMed
Google Scholar
10
P Lou, et al., Quantitative trait loci for flowering time and
morphological traits in multiple populations of Brassica rapa. J Exp
Bot 58, 4005–4016 (2007).
Crossref
PubMed
Google Scholar
11
M Debieu, et al., Co-variation between seed dormancy, growth rate
and flowering time changes with latitude in Arabidopsis thaliana.
PLoS One 8, e61075 (2013).
Crossref
PubMed
Google Scholar
12
JD Bewley, KJ Bradford, HWM Hilhorst, H Nonogaki Seeds:
Physiology of Development, Germination and Dormancy (Springer, 3rd
Ed, New York, 2013).
Crossref
Google Scholar

## Page 28

13
M Chen, et al., Maternal temperature history activates Flowering
Locus T in fruits to control progeny dormancy according to time of
year. Proc Natl Acad Sci USA 111, 18787–18792 (2014).
Crossref
PubMed
Google Scholar
14
GW Bassel, et al., Genome-wide network model capturing seed
germination reveals coordinated regulation of plant cellular phase
transitions. Proc Natl Acad Sci USA 108, 9709–9714 (2011).
Crossref
PubMed
Google Scholar
15
GC Chiang, D Barua, EM Kramer, RM Amasino, K Donohue, Major
flowering time gene, flowering locus C, regulates seed germination in
Arabidopsis thaliana. Proc Natl Acad Sci USA 106, 11661–11666
(2009).
Crossref
PubMed
Google Scholar
16
LT Burghardt, BR Edwards, K Donohue, Multiple paths to similar
germination behavior in Arabidopsis thaliana. New Phytol 209,
1301–1312. 10.1111/nph.13685 (2016).
Crossref
PubMed
Google Scholar
17
SL Kendall, et al., Induction of dormancy in Arabidopsis summer
annuals requires parallel regulation of DOG1 and hormone
metabolism by low temperature and CBF transcription factors. Plant
Cell 23, 2568–2580 (2011).
Crossref
PubMed
Google Scholar
18
K Nakabayashi, et al., The time required for dormancy release in
Arabidopsis is determined by DELAY OF GERMINATION1 protein

## Page 29

levels in freshly harvested seeds. Plant Cell 24, 2826–2838 (2012).
Crossref
PubMed
Google Scholar
19
L Bentsink, J Jowett, CJ Hanhart, M Koornneef, Cloning of DOG1, a
quantitative trait locus controlling seed dormancy in Arabidopsis.
Proc Natl Acad Sci USA 103, 17042–17047 (2006).
Crossref
PubMed
Google Scholar
20
K Graeber, et al., DELAY OF GERMINATION 1 mediates a conserved
coat-dormancy mechanism for the temperature- and gibberellindependent control of seed germination. Proc Natl Acad Sci USA 111,
E3571–E3580 (2014).
Crossref
PubMed
Google Scholar
21
M Zhao, S Yang, X Liu, K Wu, Arabidopsis histone demethylases
LDL1 and LDL2 control primary seed dormancy by regulating
DELAY OF GERMINATION 1 and ABA signaling-related genes. Front
Plant Sci 6, 159 (2015).
Crossref
PubMed
Google Scholar
22
S Atwell, et al., Genome-wide association study of 107 phenotypes
in Arabidopsis thaliana inbred lines. Nature 465, 627–631 (2010).
Crossref
PubMed
Google Scholar
23
B Brachi, et al., Linkage and association mapping of Arabidopsis
thaliana flowering time in nature. PLoS Genet 6, e1000940 (2010).
Crossref
PubMed
Google Scholar
24

## Page 30

H Huo, P Dahal, K Kunusoth, CM McCallum, KJ Bradford,
Expression of 9-cis-EPOXYCAROTENOID DIOXYGENASE4 is
essential for thermoinhibition of lettuce seed germination but not
for seed development or stress tolerance. Plant Cell 25, 884–900
(2013).
Crossref
PubMed
Google Scholar
25
J Argyris, P Dahal, E Hayashi, DW Still, KJ Bradford, Genetic
variation for lettuce seed thermoinhibition is associated with
temperature-sensitive expression of abscisic acid, gibberellin, and
ethylene biosynthesis, metabolism, and response genes. Plant Physiol
148, 926–947 (2008).
Crossref
PubMed
Google Scholar
26
D Cantliffe, Y Sung, W Nascimento, Lettuce seed germination. Hortic
Rev (Am Soc Hortic Sci) 224, 229–275 (2000).
Google Scholar
27
J Argyris, et al., A gene encoding an abscisic acid biosynthetic
enzyme (LsNCED4) collocates with the high temperature
germination locus Htg6.1 in lettuce (Lactuca sp.). Theor Appl Genet
122, 95–108 (2011).
Crossref
PubMed
Google Scholar
28
S Toh, et al., High temperature-induced abscisic acid biosynthesis
and its role in the inhibition of gibberellin action in Arabidopsis
seeds. Plant Physiol 146, 1368–1385 (2008).
Crossref
PubMed
Google Scholar
29
H Huo, KJ Bradford, Molecular and hormonal regulation of
thermoinhibition of seed germination. Advances in Plant Dormancy,
ed JV Anderson (Springer, New York), pp. 3–33 (2015).

## Page 31

Crossref
Google Scholar
30
G Wu, et al., The sequential action of miR156 and miR172 regulates
developmental timing in Arabidopsis. Cell 138, 750–759 (2009).
Crossref
PubMed
Google Scholar
31
JW Wang, B Czech, D Weigel, miR156-regulated SPL transcription
factors define an endogenous flowering pathway in Arabidopsis
thaliana. Cell 138, 738–749 (2009).
Crossref
PubMed
Google Scholar
32
D Huang, C Koh, JA Feurtado, EW Tsang, AJ Cutler, MicroRNAs
and their putative targets in Brassica napus seed maturation. BMC
Genomics 14, 140 (2013).
Crossref
PubMed
Google Scholar
33
H Nonogaki, MicroRNA gene regulation cascades during early
stages of plant development. Plant Cell Physiol 51, 1840–1846
(2010).
Crossref
PubMed
Google Scholar
34
FY Yoong, et al., Genetic variation for thermotolerance in lettuce
seed germination is associated with temperature-sensitive
regulation of ETHYLENE RESPONSE FACTOR1 (ERF1). Plant Physiol
170, 472–488 (2016).
Crossref
PubMed
Google Scholar
35
I Ashikawa, M Mori, S Nakamura, F Abe, A transgenic approach to
controlling wheat seed dormancy level by using TriticeaeDOG1-like

## Page 32

genes. Transgenic Res 23, 621–629 (2014).
Crossref
PubMed
Google Scholar
36
S Yu, et al., Gibberellin regulates the Arabidopsis floral transition
through miR156-targeted SQUAMOSA PROMOTER BINDING-LIKE
transcription factors. Plant Cell 24, 3320–3332 (2012).
Crossref
PubMed
Google Scholar
37
L Daxinger, et al., Unexpected silencing effects from T-DNA tags in
Arabidopsis. Trends Plant Sci 13, 4–6 (2008).
Crossref
PubMed
Google Scholar
38
G Wu, RS Poethig, Temporal regulation of shoot development in
Arabidopsis thaliana by miR156 and its target SPL3. Development
133, 3539–3547 (2006).
Crossref
PubMed
Google Scholar
39
M Cyrek, et al., Seed dormancy in Arabidopsis thaliana is controlled
by alternative polyadenylation of DOG1. Plant Physiol 170, 947–955
(2016).
Crossref
PubMed
Google Scholar
40
Y Wang, et al., The inhibitory effect of ABA on floral transition is
mediated by ABI5 in Arabidopsis. J Exp Bot 64, 675–684 (2013).
Crossref
PubMed
Google Scholar
41
M Xie, S Zhang, B Yu, microRNA biogenesis, degradation and
activity in plants. Cell Mol Life Sci 72, 87–99 (2015).

## Page 33

Crossref
PubMed
Google Scholar
42
S Zhang, M Xie, G Ren, B Yu, CDC5, a DNA binding protein,
positively regulates posttranscriptional processing and/or
transcription of primary microRNA transcripts. Proc Natl Acad Sci
USA 110, 17588–17593 (2013).
Crossref
PubMed
Google Scholar
43
Z Huang, S Liu, KJ Bradford, TE Huxman, DL Venable, The
contribution of germination functional traits to population
dynamics of a desert plant community. Ecology 97, 250–261 (2016).
Crossref
PubMed
Google Scholar
44
K Donohue, LT Burghardt, D Runcie, KJ Bradford, J Schmitt,
Applying developmental threshold models to evolutionary ecology.
Trends Ecol Evol 30, 66–77 (2015).
Crossref
PubMed
Google Scholar
45
H He, et al., Interaction between parental environment and
genotype affects plant and seed performance in Arabidopsis. J Exp
Bot 65, 6603–6615 (2014).
Crossref
PubMed
Google Scholar
46
FM Postma, J Ågren, Maternal environment affects the genetic basis
of seed dormancy in Arabidopsis thaliana. Mol Ecol 24, 785–797
(2015).
Crossref
PubMed
Google Scholar
47

## Page 34

K Graeber, et al., Cross-species approaches to seed dormancy and
germination: Conservation and biodiversity of ABA-regulated
mechanisms and the Brassicaceae DOG1 genes. Plant Mol Biol 73,
67–87 (2010).
Crossref
PubMed
Google Scholar
48
GC Chiang, et al., Pleiotropy in the wild: The dormancy gene DOG1
exerts cascading control on life cycles. Evolution 67, 883–893
(2013).
Crossref
PubMed
Google Scholar
49
RR de Casas, et al., Seed after-ripening and dormancy determine
adult life history independently of germination timing. New Phytol
194, 868–879 (2012).
Crossref
PubMed
Google Scholar
50
LT Burghardt, CJ Metcalf, AM Wilczek, J Schmitt, K Donohue,
Modeling the influence of genetic and environmental variation on
the expression of plant life cycles across landscapes. Am Nat 185,
212–227 (2015).
Crossref
PubMed
Google Scholar
51
Y Liu, M Koornneef, WJJ Soppe, The absence of histone H2B
monoubiquitination in the Arabidopsis hub1 (rdo4) mutant reveals a
role for chromatin remodeling in seed dormancy. Plant Cell 19,
433–444 (2007).
Crossref
PubMed
Google Scholar
52
N Tamura, et al., Isolation and characterization of high
temperature-resistant germination mutants of Arabidopsis thaliana.

## Page 35

Plant Cell Physiol 47, 1081–1094 (2006).
Crossref
PubMed
Google Scholar
53
Y Cao, Y Dai, S Cui, L Ma, Histone H2B monoubiquitination in the
chromatin of FLOWERING LOCUS C regulates flowering time in
Arabidopsis. Plant Cell 20, 2586–2602 (2008).
Crossref
PubMed
Google Scholar
54
C Lu, N Fedoroff, A mutation in the Arabidopsis HYL1 gene
encoding a dsRNA binding protein affects responses to abscisic acid,
auxin, and cytokinin. Plant Cell 12, 2351–2366 (2000).
Crossref
PubMed
Google Scholar
55
IC Bezerra, SD Michaels, FM Schomburg, RM Amasino, Lesions in
the mRNA cap-binding gene ABA HYPERSENSITIVE 1 suppress
FRIGIDA-mediated delayed flowering in Arabidopsis. Plant J 40,
112–119 (2004).
Crossref
PubMed
Google Scholar
56
M Tsuzuki, A Takeda, Y Watanabe, Recovery of dicer-like 1-late
flowering phenotype by miR172 expressed by the noncanonical
DCL4-dependent biogenesis pathway. RNA 20, 1320–1327 (2014).
Crossref
PubMed
Google Scholar
57
JH Clarke, D Tack, K Findlay, M Van Montagu, M Van Lijsebettens,
The SERRATE locus controls the formation of the early juvenile
leaves and phase length in Arabidopsis. Plant J 20, 493–501 (1999).
Crossref
PubMed
Google Scholar

## Page 36

58
S Li, X Yang, F Wu, Y He, HYL1 controls the miR156-mediated
juvenile phase of vegetative growth. J Exp Bot 63, 2787–2798
(2012).
Crossref
PubMed
Google Scholar
59
RC Martin, et al., The microRNA156 and microRNA172 gene
regulation cascades at post-germinative stages in Arabidopsis. Seed
Sci Res 20, 79–87 (2010).
Crossref
Google Scholar
60
MD Nodine, DP Bartel, MicroRNAs prevent precocious gene
expression and enable pattern formation during plant
embryogenesis. Genes Dev 24, 2678–2692 (2010).
Crossref
PubMed
Google Scholar
61
MR Willmann, AJ Mehalick, RL Packer, PD Jenik, MicroRNAs
regulate the timing of embryo maturation in Arabidopsis. Plant
Physiol 155, 1871–1884 (2011).
Crossref
PubMed
Google Scholar
62
S Laubinger, et al., Dual roles of the nuclear cap-binding complex
and SERRATE in pre-mRNA splicing and microRNA processing in
Arabidopsis thaliana. Proc Natl Acad Sci USA 105, 8795–8800
(2008).
Crossref
PubMed
Google Scholar
63
G Ren, et al., Regulation of miRNA abundance by RNA binding
protein TOUGH in Arabidopsis. Proc Natl Acad Sci USA 109, 12817–
12821 (2012).
Crossref

## Page 37

PubMed
Google Scholar
64
D Li, et al., Deep sequencing of maize small RNAs reveals a diverse
set of microRNA in dry and imbibed seeds. PLoS One 8, e55107
(2013).
Crossref
PubMed
Google Scholar
65
H Nonogaki, Seed dormancy and germination-emerging
mechanisms and new hypotheses. Front Plant Sci 5, 233 (2014).
Crossref
PubMed
Google Scholar
66
P Crevillén, et al., Epigenetic reprogramming that prevents
transgenerational inheritance of the vernalized state. Nature 515,
587–590 (2014).
Crossref
PubMed
Google Scholar
67
J Choi, et al., Resetting and regulation of FLOWERING LOCUS C
expression during Arabidopsis reproductive development. Plant J
57, 918–931 (2009).
Crossref
PubMed
Google Scholar
68
CC Sheldon, et al., Resetting of FLOWERING LOCUS C expression
after epigenetic repression by vernalization. Proc Natl Acad Sci USA
105, 2214–2219 (2008).
Crossref
PubMed
Google Scholar
69
J Bazin, et al., Targeted mRNA oxidation regulates sunflower seed
dormancy alleviation during dry after-ripening. Plant Cell 23, 2196–
2208 (2011).

## Page 38

Crossref
PubMed
Google Scholar
Information & Authors
Information
Published in
[image]
Proceedings of the National Academy of Sciences
Vol. 113 | No. 15
April 12, 2016
PubMed: 27035986
Classifications
1. Biological Sciences
2. Plant Biology
3. 
Submission history
Published online: March 28, 2016
Published in issue: April 12, 2016
Keywords
1. seed dormancy
2. flowering
3. DOG1
4. miRNA
5. lettuce
Acknowledgments

## Page 39

We thank University of California, Davis, student interns Phat Hua,
Alex Wai, and Macrae Dec-Hull and visiting Universidade Federal
de Viçosa (Brazil) student Danubia Nobre for assistance in
maintaining plants. Extramural funding for this work was provided
by United States Department of Agriculture (USDA)-National
Institute of Food and Agriculture Grant 2008-02509, USDACooperative State Research, Education, and Extension Service
Regional Research Project W3168, and Rijk Zwaan B.V., De Lier,
The Netherlands. Access to lettuce genomic sequences was provided
through the University of California, Davis, Genome Center and
supported in part by National Science Foundation Award 0820451
and the Compositae Genome Project. The visit of S.W. to University
of California, Davis, was supported by China Scholarship Council
(201203250049) and Natural Science Foundation of China
(31171867).
Notes
This article is a PNAS Direct Submission.
Authors
Affiliations
Heqiang Huo
Department of Plant Sciences, Seed Biotechnology Center,
University of California, Davis, CA 95616;
View all articles by this author
Shouhui Wei
Institute of Plant Protection, Chinese Academy of Agricultural
Sciences, Beijing 100193, China
View all articles by this author
Kent J. Bradford1 [email protected]
Department of Plant Sciences, Seed Biotechnology Center,
University of California, Davis, CA 95616;
View all articles by this author
Notes

## Page 40

1
To whom correspondence should be addressed. Email:
[email protected].
Author contributions: H.H. and K.J.B. designed research; H.H. and
S.W. performed research; H.H. and K.J.B. analyzed data; and H.H.
and K.J.B. wrote the paper.
Competing Interests
The authors declare no conflict of interest.
Metrics & Citations
 Note: The article usage is presented with a three- to four-day delay
and will update daily once available. Due to ths delay, usage data
will not appear immediately following publication. Citation
information is sourced from Crossref Cited-by service.
Metrics
Altmetrics
Citations
Cite this article
•  H. Huo,
•  S. Wei,
•  & K.J. Bradford,

## Page 41

DELAY OF GERMINATION1 (DOG1) regulates both seed dormancy
and flowering time through microRNA pathways, Proc. Natl. Acad.
Sci. U.S.A. 113 (15) E2199-E2206, https://doi.org/10.1073/
pnas.1600558113 (2016).
Copy
Copied!
Copying failed.
Export the article citation data by selecting a format from the list
below and clicking Export.
Format
Select a format to exportRIS (ProCite, Reference
Manager)EndNoteBibTexMedlarsRefWorks
Direct import
Cited by
Loading...
View Options
View options
PDF
PDF
Download this article as a PDF file.
eReader
eReader
View this article with eReader.

## Page 42

Login options
Check if you have access through your login credentials or your
institution to get full access on this article.
Personal login Institutional Login
Recommend to a librarian
Recommend PNAS to a Librarian
Save for later Item saved, go to cart
Purchase options
Purchase this article to access the full text.
Single Article Purchase
DELAY OF GERMINATION1 (DOG1) regulates both seed
dormancy and flowering time through microRNA pathways
Proceedings of the National Academy of Sciences
•  Vol. 113
•  No. 15
•  pp. 3903-E2208
$20.00
Add to Cart
Checkout
Restore content access
Restore content access for purchases made as a guest

## Page 43

Figures
Tables
Media
Share
Share
Share article link
COPY LINK
Copied!
Copying failed.
Share on social media
FacebookX (formerly Twitter)LinkedInGmailemail
View full text|Download PDF
NOW READING:
Share
•  
•  
•  
•  
•

## Page 44

Further reading in this issue
•  Research ArticleMarch 24, 2016
Modulation of nitrogen vacancy charge state and
fluorescence in nanodiamonds using
electrochemical potential
○  Sinan Karaveli,
○  Ophir Gaathon,
○  Abraham Wolcott,
○  Reyu Sakakibara,
○  Or A. Shemesh,
○  Darcy S. Peterka,
○  Edward S. Boyden,
○  Jonathan S. Owen,
○  Rafael Yuste,
○  Dirk Englund,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
•  Research ArticleMarch 28, 2016
Discrete gene replication events drive coupling
between the cell cycle and circadian clocks
○  Joris Paijmans,
○  Mark Bosman,
○  Pieter Rein ten Wolde,
○  David K. Lubensky,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
•  Research ArticleMarch 28, 2016
Dominant, open nonverbal displays are
attractive at zero-acquaintance

## Page 45

○  Tanya Vacharkulksemsuk,
○  Emily Reit,
○  Poruz Khambatta,
○  Paul W. Eastwick,
○  Eli J. Finkel,
○  Dana R. Carney,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
Trending
•  Research ArticleSeptember 16, 2025
Life without sex: Large-scale study links
sexlessness to physical, cognitive, and
personality traits, socioecological factors, and
DNA
Sexual partnerships can have a profound impact on societal
well-being and evolution. Their absence can be detrimental to
mental health and can lead to behavioral problems.
Importantly, lifetime sexlessness offers an insightful measure
for ...Romantic (typically sexual) relationships are important
to personal, physical, mental, social, and economic wellbeing, and to human evolution. Yet little is known about
factors contributing to long-term lack of intimate
relationships. We investigated ...
○  Abdel Abdellaoui,
○  Laura W. Wesseldijk,
○  Scott D. Gordon,
○  Joëlle A. Pasman,
○  Dirk J. A. Smit,
○  Renáta Androvičová,
○  Nicholas G. Martin,
○  Fredrik Ullén,
○  Miriam A. Mosing,
○  Brendan P. Zietsch,

## Page 46

○  Karin J. H. Verweij,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
•  Research ArticleDecember 30, 2013
Bodily maps of emotions
Emotions coordinate our behavior and physiological states
during survival-salient events and pleasurable interactions.
Even though we are often consciously aware of our current
emotional state, such as anger or happiness, the mechanisms
giving ...Emotions are often felt in the body, and
somatosensory feedback has been proposed to trigger
conscious emotional experiences. Here we reveal maps of
bodily sensations associated with different emotions using a
unique topographical self-report method. In ...
○  Lauri Nummenmaa,
○  Enrico Glerean,
○  Riitta Hari,
○  Jari K. Hietanen,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
•  PerspectiveOctober 6, 2025
Generative AI for computational chemistry: A
roadmap to predicting emergent phenomena
The recent surge in generative AI has introduced exciting
possibilities for computational chemistry. Generative AI
methods have made significant progress in sampling
molecular structures across chemical species, developing
force fields, and speeding up ...
○  Pratyush Tiwary,
○  Lukas Herron,

## Page 47

○  Richard John,
○  Suemin Lee,
○  Disha Sanwal,
○  Ruiyu Wang,
This is an addendum toComplex viscosity of helical and
doubly helical polymeric liquids from general rigid bead-rod
theory
Sign up for thePNAS Highlights newsletter
Get in-depth science stories sent to your inbox twice a month.
Email address Subscribe
[image]
•  
•  
•  
•  
•  
•  
•  
Browse
•  Current Issue
•  PNAS Nexus
•  Special Features
•  List of Issues
•  Topics, Collections, and Article Types
•  PNAS In The News
•  Front Matter
•  Journal Club
•  Multimedia
•  Podcasts
Information
•  About

## Page 48

•  Sustainable Development Goals
•  Editorial Board
•  Authors
•  Reviewers
•  Subscribers
•  Librarians
•  Press
•  Cozzarelli Prize
•  PNAS Updates
Back to top
Copyright © 2025 National Academy of Sciences. All rights
reserved. | Online ISSN 1091-6490
PNAS is a partner of CHORUS, CLOCKSS, COPE, CrossRef, ORCID,
and Research4Life.
•  Contact
•  Site Map
•  Terms & Privacy Policy
•  Accessibility
Request Username
x Close
Can't sign in? Forgot your username?
Enter your email address below and we will send you your
username
Email
Submit
If the address matches an existing account you will receive an email
with instructions to retrieve your username
Create a new account

## Page 49

xClose
Email
Returning user
Submit
CONTINUE
We use cookies on this site to enhance your user experience. By
using this website, you are giving your consent for us to set cookies.
Find out more
Login
xClose
Email*
Password*
Forgot your password? Reset it here
Keep me logged in
Login
Don’t have an account? Register here
Institutional Login
Change Password
xClose
Enter the verification code
Submit
Congrats!
Your Phone has been verified

